apps/asn1pars.c

```diff
@@ -85,7 +85,7 @@ int asn1parse_main(int argc, char **argv)
     }
 
     while ((o = opt_next()) != OPT_EOF) {
-        switch (o) {
+        switch ((OPTION_CHOICE)o) {
```

Comment:
This sort of cast makes no sense at all.  You have already defined `o` with that type, so why the cast?

Reply:
`opt_int` is declared to return an int.  By casting in the switch, the compiler can find missing cases for you.

Try this experiment
- Take your master branch, edit apps/enc.c (or command) and remove the `OPT_EOF` case. Build.  No warning.
- Now add the cast (keep the removal you just did).  Build.  Warning about unhandled value.

Reply:
What does it matter what `opt_int` (I assume you meant `opt_next`) returns?  *`o` is defined as an `OPTION_CHOICE`, so the cast is 100% unnecessary!*  See line 77.

As for experiment:

``` diff
diff --git a/apps/enc.c b/apps/enc.c
index 242d3ef0aa..54b4453c20 100644
--- a/apps/enc.c
+++ b/apps/enc.c
@@ -146,7 +146,6 @@ int enc_main(int argc, char **argv)
     prog = opt_init(argc, argv, enc_options);
     while ((o = opt_next()) != OPT_EOF) {
         switch (o) {
-        case OPT_EOF:
         case OPT_ERR:
  opthelp:
             BIO_printf(bio_err, "%s: Use -help for summary.\n", prog);
```

``` console
gcc  -Iapps -I. -Iinclude -Iapps/include -I../master -I../master/include -I../master/apps/include  -pthread -m64 -Wa,--noexecstack -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -DOPENSSL_BUILDING_OPENSSL  -MMD -MF apps/openssl-bin-enc.d.tmp -MT apps/openssl-bin-enc.o -c -o apps/openssl-bin-enc.o ../master/apps/enc.c
../master/apps/enc.c: In function 'enc_main':
../master/apps/enc.c:148:9: error: enumeration value 'OPT_EOF' not handled in switch [-Werror=switch]
  148 |         switch (o) {
      |         ^~~~~~
cc1: all warnings being treated as errors
make: *** [Makefile:23826: apps/openssl-bin-enc.o] Fel 1
```

So, uhm, come again?

Reply:
Some compilers don't.  For example:
```
; o

        ; cd /home/rsalz/openssl/github
On branch doc-opt-api
Untracked files:
  (use "git add <file>..." to include in what will be committed)

	a.c

nothing added to commit but untracked files present (use "git add" to track)
; g b
  OpenSSL_1_1_1-stable 86a90dc749 [origin/OpenSSL_1_1_1-stable: behind 7] Handle set_alpn_protos inputs better.
  alt-14970            87ae05c352 [github/alt-14970] Add 'N' option type, have cms use it
  build-apps-tests     0ebf3e387d Add script to create an alternate Makefile
  dc                   dacf0acc9e Checkpoint
  deprecate-comp       0cf1f2171e Deprecate the COMP_xxx API
* doc-opt-api          3efb0a5f0e fixup! Fix nits, add some text.
  fix-wrap-opt         c110f5af73 Fetch cipher-wrap after loading providers.
  master               a70936a845 [origin/master] TEST: correct test/recipes/30-test_evp_data/evppkey_ecdh.txt
  mv-prov-names-hdr    bbfa4114a1 Make prov/names.h a public header file openssl/prov_names.h
  switch-safety        fae8ad4e99 Case to an OPTION insie the switch()
; g co switch-safety
Switched to branch 'switch-safety'
; vi apps/enc.c

Detected changed: Configure Configurations/unix-Makefile.tmpl
/usr/bin/perl configdata.pm -r
Configuring OpenSSL version 3.0.0-alpha16-dev for target linux-x86_64
Using os-specific seed configuration
Creating configdata.pm
Running configdata.pm
Creating Makefile

**********************************************************************
***                                                                ***
***   OpenSSL has been successfully configured                     ***
***                                                                ***
***   If you encounter a problem while building, please open an    ***
***   issue on GitHub <https://github.com/openssl/openssl/issues>  ***
***   and include the output from the following command:           ***
***                                                                ***
***       perl configdata.pm --dump                                ***
***                                                                ***
***   (If you are new to OpenSSL, you might want to consult the    ***
***   'Troubleshooting' section in the INSTALL.md file first)      ***
***                                                                ***
**********************************************************************
**************************************************
***                                            ***
***   Please run the same make command again   ***
***                                            ***
**************************************************
Makefile:4105: recipe for target 'configdata.pm' failed
make: *** [configdata.pm] Error 1

Press ENTER or type command to continue
; mk
/usr/bin/perl "-I." -Mconfigdata "util/dofile.pl" "-oMakefile" include/crypto/bn_conf.h.in > include/crypto/bn_conf.h
/usr/bin/perl "-I." -Mconfigdata "util/dofile.pl" "-oMakefile" include/crypto/dso_conf.h.in > include/crypto/dso_conf.h
/usr/bin/perl "-I." -Mconfigdata "util/dofile.pl" "-oMakefile" include/openssl/asn1.h.in > include/openssl/asn1.h
/usr/bin/perl "-I." -Mconfigdata "util/dofile.pl" "-oMakefile" include/openssl/asn1t.h.in > include/openssl/asn1t.h
/usr/bin/perl "-I." -Mconfigdata "util/dofile.pl" "-oMakefile" include/openssl/bio.h.in > include/openssl/bio.h
/usr/bin/perl "-I." -Mconfigdata "util/dofile.pl" "-oMakefile" include/openssl/cmp.h.in > include/openssl/cmp.h
/usr/bin/perl "-I." -Mconfigdata "util/dofile.pl" "-oMakefile" include/openssl/cms.h.in > include/openssl/cms.h
/usr/bin/perl "-I." -Mconfigdata "util/dofile.pl" "-oMakefile" include/openssl/conf.h.in > include/openssl/conf.h
/usr/bin/perl "-I." -Mconfigdata "util/dofile.pl" "-oMakefile" include/openssl/configuration.h.in > include/openssl/configuration.h
/usr/bin/perl "-I." -Mconfigdata "util/dofile.pl" "-oMakefile" include/openssl/crmf.h.in > include/openssl/crmf.h
/usr/bin/perl "-I." -Mconfigdata "util/dofile.pl" "-oMakefile" include/openssl/crypto.h.in > include/openssl/crypto.h
/usr/bin/perl "-I." -Mconfigdata "util/dofile.pl" "-oMakefile" include/openssl/ct.h.in > include/openssl/ct.h
/usr/bin/perl "-I." -Mconfigdata "util/dofile.pl" "-oMakefile" include/openssl/err.h.in > include/openssl/err.h
/usr/bin/perl "-I." -Mconfigdata "util/dofile.pl" "-oMakefile" include/openssl/ess.h.in > include/openssl/ess.h
/usr/bin/perl "-I." -Mconfigdata "util/dofile.pl" "-oMakefile" include/openssl/fipskey.h.in > include/openssl/fipskey.h
/usr/bin/perl "-I." -Mconfigdata "util/dofile.pl" "-oMakefile" include/openssl/lhash.h.in > include/openssl/lhash.h
/usr/bin/perl "-I." -Mconfigdata "util/dofile.pl" "-oMakefile" include/openssl/ocsp.h.in > include/openssl/ocsp.h
/usr/bin/perl "-I." -Mconfigdata "util/dofile.pl" "-oMakefile" include/openssl/opensslv.h.in > include/openssl/opensslv.h
/usr/bin/perl "-I." -Mconfigdata "util/dofile.pl" "-oMakefile" include/openssl/pkcs12.h.in > include/openssl/pkcs12.h
/usr/bin/perl "-I." -Mconfigdata "util/dofile.pl" "-oMakefile" include/openssl/pkcs7.h.in > include/openssl/pkcs7.h
/usr/bin/perl "-I." -Mconfigdata "util/dofile.pl" "-oMakefile" include/openssl/safestack.h.in > include/openssl/safestack.h
/usr/bin/perl "-I." -Mconfigdata "util/dofile.pl" "-oMakefile" include/openssl/srp.h.in > include/openssl/srp.h
/usr/bin/perl "-I." -Mconfigdata "util/dofile.pl" "-oMakefile" include/openssl/ssl.h.in > include/openssl/ssl.h
/usr/bin/perl "-I." -Mconfigdata "util/dofile.pl" "-oMakefile" include/openssl/ui.h.in > include/openssl/ui.h
/usr/bin/perl "-I." -Mconfigdata "util/dofile.pl" "-oMakefile" include/openssl/x509.h.in > include/openssl/x509.h
/usr/bin/perl "-I." -Mconfigdata "util/dofile.pl" "-oMakefile" include/openssl/x509_vfy.h.in > include/openssl/x509_vfy.h
/usr/bin/perl "-I." -Mconfigdata "util/dofile.pl" "-oMakefile" include/openssl/x509v3.h.in > include/openssl/x509v3.h
/usr/bin/perl "-I." -Mconfigdata "util/dofile.pl" "-oMakefile" test/provider_internal_test.cnf.in > test/provider_internal_test.cnf
make depend && make _build_sw
make[1]: Entering directory '/home/rsalz/openssl/github'
make[1]: Leaving directory '/home/rsalz/openssl/github'
make[1]: Entering directory '/home/rsalz/openssl/github'
ccache clang  -I. -Iinclude -Iapps/include  -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF apps/lib/libapps-lib-opt.d.tmp -MT apps/lib/libapps-lib-opt.o -c -o apps/lib/libapps-lib-opt.o apps/lib/opt.c
rm -f apps/libapps.a
ar qc apps/libapps.a apps/lib/libapps-lib-app_params.o apps/lib/libapps-lib-app_provider.o apps/lib/libapps-lib-app_rand.o apps/lib/libapps-lib-app_x509.o apps/lib/libapps-lib-apps.o apps/lib/libapps-lib-apps_ui.o apps/lib/libapps-lib-columns.o apps/lib/libapps-lib-engine.o apps/lib/libapps-lib-engine_loader.o apps/lib/libapps-lib-fmt.o apps/lib/libapps-lib-http_server.o apps/lib/libapps-lib-names.o apps/lib/libapps-lib-opt.o apps/lib/libapps-lib-s_cb.o apps/lib/libapps-lib-s_socket.o apps/lib/libapps-lib-tlssrp_depr.o
ranlib apps/libapps.a || echo Never mind.
ccache clang  -I. -Iinclude -Iproviders/common/include -Iproviders/implementations/include  -DAES_ASM -DBSAES_ASM -DCMLL_ASM -DECP_NISTZ256_ASM -DGHASH_ASM -DKECCAK1600_ASM -DMD5_ASM -DOPENSSL_BN_ASM_GF2m -DOPENSSL_BN_ASM_MONT -DOPENSSL_BN_ASM_MONT5 -DOPENSSL_CPUID_OBJ -DOPENSSL_IA32_SSE2 -DPOLY1305_ASM -DSHA1_ASM -DSHA256_ASM -DSHA512_ASM -DVPAES_ASM -DWHIRLPOOL_ASM -DX25519_ASM -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF crypto/evp/libcrypto-lib-keymgmt_lib.d.tmp -MT crypto/evp/libcrypto-lib-keymgmt_lib.o -c -o crypto/evp/libcrypto-lib-keymgmt_lib.o crypto/evp/keymgmt_lib.c
ccache clang  -I. -Iinclude -Iproviders/common/include -Iproviders/implementations/include  -DAES_ASM -DBSAES_ASM -DCMLL_ASM -DECP_NISTZ256_ASM -DGHASH_ASM -DKECCAK1600_ASM -DMD5_ASM -DOPENSSL_BN_ASM_GF2m -DOPENSSL_BN_ASM_MONT -DOPENSSL_BN_ASM_MONT5 -DOPENSSL_CPUID_OBJ -DOPENSSL_IA32_SSE2 -DPOLY1305_ASM -DSHA1_ASM -DSHA256_ASM -DSHA512_ASM -DVPAES_ASM -DWHIRLPOOL_ASM -DX25519_ASM -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF crypto/evp/libcrypto-lib-p_lib.d.tmp -MT crypto/evp/libcrypto-lib-p_lib.o -c -o crypto/evp/libcrypto-lib-p_lib.o crypto/evp/p_lib.c
ccache clang  -I. -Iinclude -Iproviders/common/include -Iproviders/implementations/include  -DAES_ASM -DBSAES_ASM -DCMLL_ASM -DECP_NISTZ256_ASM -DGHASH_ASM -DKECCAK1600_ASM -DMD5_ASM -DOPENSSL_BN_ASM_GF2m -DOPENSSL_BN_ASM_MONT -DOPENSSL_BN_ASM_MONT5 -DOPENSSL_CPUID_OBJ -DOPENSSL_IA32_SSE2 -DPOLY1305_ASM -DSHA1_ASM -DSHA256_ASM -DSHA512_ASM -DVPAES_ASM -DWHIRLPOOL_ASM -DX25519_ASM -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF crypto/store/libcrypto-lib-store_result.d.tmp -MT crypto/store/libcrypto-lib-store_result.o -c -o crypto/store/libcrypto-lib-store_result.o crypto/store/store_result.c
/usr/bin/perl "-I." "-Iproviders/common/der" -Mconfigdata -Moids_to_c "util/dofile.pl" "-oMakefile" providers/common/der/der_rsa.h.in > providers/common/include/prov/der_rsa.h
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -DAES_ASM -DBSAES_ASM -DECP_NISTZ256_ASM -DGHASH_ASM -DKECCAK1600_ASM -DMD5_ASM -DOPENSSL_BN_ASM_GF2m -DOPENSSL_BN_ASM_MONT -DOPENSSL_BN_ASM_MONT5 -DOPENSSL_CPUID_OBJ -DOPENSSL_IA32_SSE2 -DPOLY1305_ASM -DSHA1_ASM -DSHA256_ASM -DSHA512_ASM -DVPAES_ASM -DWHIRLPOOL_ASM -DX25519_ASM -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/implementations/encode_decode/libimplementations-lib-encode_key2any.d.tmp -MT providers/implementations/encode_decode/libimplementations-lib-encode_key2any.o -c -o providers/implementations/encode_decode/libimplementations-lib-encode_key2any.o providers/implementations/encode_decode/encode_key2any.c
/usr/bin/perl "-I." "-Iproviders/common/der" -Mconfigdata -Moids_to_c "util/dofile.pl" "-oMakefile" providers/common/der/der_wrap.h.in > providers/common/include/prov/der_wrap.h
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -DAES_ASM -DBSAES_ASM -DECP_NISTZ256_ASM -DGHASH_ASM -DKECCAK1600_ASM -DMD5_ASM -DOPENSSL_BN_ASM_GF2m -DOPENSSL_BN_ASM_MONT -DOPENSSL_BN_ASM_MONT5 -DOPENSSL_CPUID_OBJ -DOPENSSL_IA32_SSE2 -DPOLY1305_ASM -DSHA1_ASM -DSHA256_ASM -DSHA512_ASM -DVPAES_ASM -DWHIRLPOOL_ASM -DX25519_ASM -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/implementations/kdfs/libimplementations-lib-x942kdf.d.tmp -MT providers/implementations/kdfs/libimplementations-lib-x942kdf.o -c -o providers/implementations/kdfs/libimplementations-lib-x942kdf.o providers/implementations/kdfs/x942kdf.c
/usr/bin/perl "-I." "-Iproviders/common/der" -Mconfigdata -Moids_to_c "util/dofile.pl" "-oMakefile" providers/common/der/der_dsa.h.in > providers/common/include/prov/der_dsa.h
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -DAES_ASM -DBSAES_ASM -DECP_NISTZ256_ASM -DGHASH_ASM -DKECCAK1600_ASM -DMD5_ASM -DOPENSSL_BN_ASM_GF2m -DOPENSSL_BN_ASM_MONT -DOPENSSL_BN_ASM_MONT5 -DOPENSSL_CPUID_OBJ -DOPENSSL_IA32_SSE2 -DPOLY1305_ASM -DSHA1_ASM -DSHA256_ASM -DSHA512_ASM -DVPAES_ASM -DWHIRLPOOL_ASM -DX25519_ASM -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/implementations/signature/libimplementations-lib-dsa.d.tmp -MT providers/implementations/signature/libimplementations-lib-dsa.o -c -o providers/implementations/signature/libimplementations-lib-dsa.o providers/implementations/signature/dsa.c
/usr/bin/perl "-I." "-Iproviders/common/der" -Mconfigdata -Moids_to_c "util/dofile.pl" "-oMakefile" providers/common/der/der_ec.h.in > providers/common/include/prov/der_ec.h
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -DAES_ASM -DBSAES_ASM -DECP_NISTZ256_ASM -DGHASH_ASM -DKECCAK1600_ASM -DMD5_ASM -DOPENSSL_BN_ASM_GF2m -DOPENSSL_BN_ASM_MONT -DOPENSSL_BN_ASM_MONT5 -DOPENSSL_CPUID_OBJ -DOPENSSL_IA32_SSE2 -DPOLY1305_ASM -DSHA1_ASM -DSHA256_ASM -DSHA512_ASM -DVPAES_ASM -DWHIRLPOOL_ASM -DX25519_ASM -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/implementations/signature/libimplementations-lib-ecdsa.d.tmp -MT providers/implementations/signature/libimplementations-lib-ecdsa.o -c -o providers/implementations/signature/libimplementations-lib-ecdsa.o providers/implementations/signature/ecdsa.c
/usr/bin/perl "-I." "-Iproviders/common/der" -Mconfigdata -Moids_to_c "util/dofile.pl" "-oMakefile" providers/common/der/der_ecx.h.in > providers/common/include/prov/der_ecx.h
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -DAES_ASM -DBSAES_ASM -DECP_NISTZ256_ASM -DGHASH_ASM -DKECCAK1600_ASM -DMD5_ASM -DOPENSSL_BN_ASM_GF2m -DOPENSSL_BN_ASM_MONT -DOPENSSL_BN_ASM_MONT5 -DOPENSSL_CPUID_OBJ -DOPENSSL_IA32_SSE2 -DPOLY1305_ASM -DSHA1_ASM -DSHA256_ASM -DSHA512_ASM -DVPAES_ASM -DWHIRLPOOL_ASM -DX25519_ASM -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/implementations/signature/libimplementations-lib-eddsa.d.tmp -MT providers/implementations/signature/libimplementations-lib-eddsa.o -c -o providers/implementations/signature/libimplementations-lib-eddsa.o providers/implementations/signature/eddsa.c
/usr/bin/perl "-I." "-Iproviders/common/der" -Mconfigdata -Moids_to_c "util/dofile.pl" "-oMakefile" providers/common/der/der_sm2.h.in > providers/common/include/prov/der_sm2.h
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -DAES_ASM -DBSAES_ASM -DECP_NISTZ256_ASM -DGHASH_ASM -DKECCAK1600_ASM -DMD5_ASM -DOPENSSL_BN_ASM_GF2m -DOPENSSL_BN_ASM_MONT -DOPENSSL_BN_ASM_MONT5 -DOPENSSL_CPUID_OBJ -DOPENSSL_IA32_SSE2 -DPOLY1305_ASM -DSHA1_ASM -DSHA256_ASM -DSHA512_ASM -DVPAES_ASM -DWHIRLPOOL_ASM -DX25519_ASM -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/implementations/signature/libimplementations-lib-sm2sig.d.tmp -MT providers/implementations/signature/libimplementations-lib-sm2sig.o -c -o providers/implementations/signature/libimplementations-lib-sm2sig.o providers/implementations/signature/sm2sig.c
ccache clang  -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -DAES_ASM -DBSAES_ASM -DECP_NISTZ256_ASM -DGHASH_ASM -DKECCAK1600_ASM -DMD5_ASM -DOPENSSL_BN_ASM_GF2m -DOPENSSL_BN_ASM_MONT -DOPENSSL_BN_ASM_MONT5 -DOPENSSL_CPUID_OBJ -DOPENSSL_IA32_SSE2 -DPOLY1305_ASM -DSHA1_ASM -DSHA256_ASM -DSHA512_ASM -DVPAES_ASM -DWHIRLPOOL_ASM -DX25519_ASM -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/implementations/storemgmt/libimplementations-lib-file_store_der2obj.d.tmp -MT providers/implementations/storemgmt/libimplementations-lib-file_store_der2obj.o -c -o providers/implementations/storemgmt/libimplementations-lib-file_store_der2obj.o providers/implementations/storemgmt/file_store_der2obj.c
/usr/bin/perl "-I." "-Iproviders/common/der" -Mconfigdata -Moids_to_c "util/dofile.pl" "-oMakefile" providers/common/der/der_digests_gen.c.in > providers/common/der/der_digests_gen.c
/usr/bin/perl "-I." "-Iproviders/common/der" -Mconfigdata -Moids_to_c "util/dofile.pl" "-oMakefile" providers/common/der/der_digests.h.in > providers/common/include/prov/der_digests.h
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/common/der/libnonfips-lib-der_digests_gen.d.tmp -MT providers/common/der/libnonfips-lib-der_digests_gen.o -c -o providers/common/der/libnonfips-lib-der_digests_gen.o providers/common/der/der_digests_gen.c
/usr/bin/perl "-I." "-Iproviders/common/der" -Mconfigdata -Moids_to_c "util/dofile.pl" "-oMakefile" providers/common/der/der_dsa_gen.c.in > providers/common/der/der_dsa_gen.c
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/common/der/libnonfips-lib-der_dsa_gen.d.tmp -MT providers/common/der/libnonfips-lib-der_dsa_gen.o -c -o providers/common/der/libnonfips-lib-der_dsa_gen.o providers/common/der/der_dsa_gen.c
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/common/der/libnonfips-lib-der_dsa_key.d.tmp -MT providers/common/der/libnonfips-lib-der_dsa_key.o -c -o providers/common/der/libnonfips-lib-der_dsa_key.o providers/common/der/der_dsa_key.c
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/common/der/libnonfips-lib-der_dsa_sig.d.tmp -MT providers/common/der/libnonfips-lib-der_dsa_sig.o -c -o providers/common/der/libnonfips-lib-der_dsa_sig.o providers/common/der/der_dsa_sig.c
/usr/bin/perl "-I." "-Iproviders/common/der" -Mconfigdata -Moids_to_c "util/dofile.pl" "-oMakefile" providers/common/der/der_ec_gen.c.in > providers/common/der/der_ec_gen.c
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/common/der/libnonfips-lib-der_ec_gen.d.tmp -MT providers/common/der/libnonfips-lib-der_ec_gen.o -c -o providers/common/der/libnonfips-lib-der_ec_gen.o providers/common/der/der_ec_gen.c
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/common/der/libnonfips-lib-der_ec_key.d.tmp -MT providers/common/der/libnonfips-lib-der_ec_key.o -c -o providers/common/der/libnonfips-lib-der_ec_key.o providers/common/der/der_ec_key.c
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/common/der/libnonfips-lib-der_ec_sig.d.tmp -MT providers/common/der/libnonfips-lib-der_ec_sig.o -c -o providers/common/der/libnonfips-lib-der_ec_sig.o providers/common/der/der_ec_sig.c
/usr/bin/perl "-I." "-Iproviders/common/der" -Mconfigdata -Moids_to_c "util/dofile.pl" "-oMakefile" providers/common/der/der_ecx_gen.c.in > providers/common/der/der_ecx_gen.c
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/common/der/libnonfips-lib-der_ecx_gen.d.tmp -MT providers/common/der/libnonfips-lib-der_ecx_gen.o -c -o providers/common/der/libnonfips-lib-der_ecx_gen.o providers/common/der/der_ecx_gen.c
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/common/der/libnonfips-lib-der_ecx_key.d.tmp -MT providers/common/der/libnonfips-lib-der_ecx_key.o -c -o providers/common/der/libnonfips-lib-der_ecx_key.o providers/common/der/der_ecx_key.c
/usr/bin/perl "-I." "-Iproviders/common/der" -Mconfigdata -Moids_to_c "util/dofile.pl" "-oMakefile" providers/common/der/der_rsa_gen.c.in > providers/common/der/der_rsa_gen.c
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/common/der/libnonfips-lib-der_rsa_gen.d.tmp -MT providers/common/der/libnonfips-lib-der_rsa_gen.o -c -o providers/common/der/libnonfips-lib-der_rsa_gen.o providers/common/der/der_rsa_gen.c
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/common/der/libnonfips-lib-der_rsa_key.d.tmp -MT providers/common/der/libnonfips-lib-der_rsa_key.o -c -o providers/common/der/libnonfips-lib-der_rsa_key.o providers/common/der/der_rsa_key.c
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/common/der/libnonfips-lib-der_rsa_sig.d.tmp -MT providers/common/der/libnonfips-lib-der_rsa_sig.o -c -o providers/common/der/libnonfips-lib-der_rsa_sig.o providers/common/der/der_rsa_sig.c
/usr/bin/perl "-I." "-Iproviders/common/der" -Mconfigdata -Moids_to_c "util/dofile.pl" "-oMakefile" providers/common/der/der_sm2_gen.c.in > providers/common/der/der_sm2_gen.c
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/common/der/libnonfips-lib-der_sm2_gen.d.tmp -MT providers/common/der/libnonfips-lib-der_sm2_gen.o -c -o providers/common/der/libnonfips-lib-der_sm2_gen.o providers/common/der/der_sm2_gen.c
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/common/der/libnonfips-lib-der_sm2_key.d.tmp -MT providers/common/der/libnonfips-lib-der_sm2_key.o -c -o providers/common/der/libnonfips-lib-der_sm2_key.o providers/common/der/der_sm2_key.c
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/common/der/libnonfips-lib-der_sm2_sig.d.tmp -MT providers/common/der/libnonfips-lib-der_sm2_sig.o -c -o providers/common/der/libnonfips-lib-der_sm2_sig.o providers/common/der/der_sm2_sig.c
/usr/bin/perl "-I." "-Iproviders/common/der" -Mconfigdata -Moids_to_c "util/dofile.pl" "-oMakefile" providers/common/der/der_wrap_gen.c.in > providers/common/der/der_wrap_gen.c
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/common/der/libnonfips-lib-der_wrap_gen.d.tmp -MT providers/common/der/libnonfips-lib-der_wrap_gen.o -c -o providers/common/der/libnonfips-lib-der_wrap_gen.o providers/common/der/der_wrap_gen.c
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/implementations/signature/libnonfips-lib-rsa.d.tmp -MT providers/implementations/signature/libnonfips-lib-rsa.o -c -o providers/implementations/signature/libnonfips-lib-rsa.o providers/implementations/signature/rsa.c
rm -f libcrypto.a
ar qc libcrypto.a crypto/aes/libcrypto-lib-aes-x86_64.o crypto/aes/libcrypto-lib-aes_cfb.o crypto/aes/libcrypto-lib-aes_ecb.o crypto/aes/libcrypto-lib-aes_ige.o crypto/aes/libcrypto-lib-aes_misc.o crypto/aes/libcrypto-lib-aes_ofb.o crypto/aes/libcrypto-lib-aes_wrap.o crypto/aes/libcrypto-lib-aesni-mb-x86_64.o crypto/aes/libcrypto-lib-aesni-sha1-x86_64.o crypto/aes/libcrypto-lib-aesni-sha256-x86_64.o crypto/aes/libcrypto-lib-aesni-x86_64.o crypto/aes/libcrypto-lib-bsaes-x86_64.o crypto/aes/libcrypto-lib-vpaes-x86_64.o crypto/aria/libcrypto-lib-aria.o crypto/asn1/libcrypto-lib-a_bitstr.o crypto/asn1/libcrypto-lib-a_d2i_fp.o crypto/asn1/libcrypto-lib-a_digest.o crypto/asn1/libcrypto-lib-a_dup.o crypto/asn1/libcrypto-lib-a_gentm.o crypto/asn1/libcrypto-lib-a_i2d_fp.o crypto/asn1/libcrypto-lib-a_int.o crypto/asn1/libcrypto-lib-a_mbstr.o crypto/asn1/libcrypto-lib-a_object.o crypto/asn1/libcrypto-lib-a_octet.o crypto/asn1/libcrypto-lib-a_print.o crypto/asn1/libcrypto-lib-a_sign.o crypto/asn1/libcrypto-lib-a_strex.o crypto/asn1/libcrypto-lib-a_strnid.o crypto/asn1/libcrypto-lib-a_time.o crypto/asn1/libcrypto-lib-a_type.o crypto/asn1/libcrypto-lib-a_utctm.o crypto/asn1/libcrypto-lib-a_utf8.o crypto/asn1/libcrypto-lib-a_verify.o crypto/asn1/libcrypto-lib-ameth_lib.o crypto/asn1/libcrypto-lib-asn1_err.o crypto/asn1/libcrypto-lib-asn1_gen.o crypto/asn1/libcrypto-lib-asn1_item_list.o crypto/asn1/libcrypto-lib-asn1_lib.o crypto/asn1/libcrypto-lib-asn1_par.o crypto/asn1/libcrypto-lib-asn_mime.o crypto/asn1/libcrypto-lib-asn_moid.o crypto/asn1/libcrypto-lib-asn_mstbl.o crypto/asn1/libcrypto-lib-asn_pack.o crypto/asn1/libcrypto-lib-bio_asn1.o crypto/asn1/libcrypto-lib-bio_ndef.o crypto/asn1/libcrypto-lib-d2i_param.o crypto/asn1/libcrypto-lib-d2i_pr.o crypto/asn1/libcrypto-lib-d2i_pu.o crypto/asn1/libcrypto-lib-evp_asn1.o crypto/asn1/libcrypto-lib-f_int.o crypto/asn1/libcrypto-lib-f_string.o crypto/asn1/libcrypto-lib-i2d_evp.o crypto/asn1/libcrypto-lib-n_pkey.o crypto/asn1/libcrypto-lib-nsseq.o crypto/asn1/libcrypto-lib-p5_pbe.o crypto/asn1/libcrypto-lib-p5_pbev2.o crypto/asn1/libcrypto-lib-p5_scrypt.o crypto/asn1/libcrypto-lib-p8_pkey.o crypto/asn1/libcrypto-lib-t_bitst.o crypto/asn1/libcrypto-lib-t_pkey.o crypto/asn1/libcrypto-lib-t_spki.o crypto/asn1/libcrypto-lib-tasn_dec.o crypto/asn1/libcrypto-lib-tasn_enc.o crypto/asn1/libcrypto-lib-tasn_fre.o crypto/asn1/libcrypto-lib-tasn_new.o crypto/asn1/libcrypto-lib-tasn_prn.o crypto/asn1/libcrypto-lib-tasn_scn.o crypto/asn1/libcrypto-lib-tasn_typ.o crypto/asn1/libcrypto-lib-tasn_utl.o crypto/asn1/libcrypto-lib-x_algor.o crypto/asn1/libcrypto-lib-x_bignum.o crypto/asn1/libcrypto-lib-x_info.o crypto/asn1/libcrypto-lib-x_int64.o crypto/asn1/libcrypto-lib-x_long.o crypto/asn1/libcrypto-lib-x_pkey.o crypto/asn1/libcrypto-lib-x_sig.o crypto/asn1/libcrypto-lib-x_spki.o crypto/asn1/libcrypto-lib-x_val.o crypto/async/arch/libcrypto-lib-async_null.o crypto/async/arch/libcrypto-lib-async_posix.o crypto/async/arch/libcrypto-lib-async_win.o crypto/async/libcrypto-lib-async.o crypto/async/libcrypto-lib-async_err.o crypto/async/libcrypto-lib-async_wait.o crypto/bf/libcrypto-lib-bf_cfb64.o crypto/bf/libcrypto-lib-bf_ecb.o crypto/bf/libcrypto-lib-bf_enc.o crypto/bf/libcrypto-lib-bf_ofb64.o crypto/bf/libcrypto-lib-bf_skey.o crypto/bio/libcrypto-lib-b_addr.o crypto/bio/libcrypto-lib-b_dump.o crypto/bio/libcrypto-lib-b_print.o crypto/bio/libcrypto-lib-b_sock.o crypto/bio/libcrypto-lib-b_sock2.o crypto/bio/libcrypto-lib-bf_buff.o crypto/bio/libcrypto-lib-bf_lbuf.o crypto/bio/libcrypto-lib-bf_nbio.o crypto/bio/libcrypto-lib-bf_null.o crypto/bio/libcrypto-lib-bf_prefix.o crypto/bio/libcrypto-lib-bf_readbuff.o crypto/bio/libcrypto-lib-bio_cb.o crypto/bio/libcrypto-lib-bio_err.o crypto/bio/libcrypto-lib-bio_lib.o crypto/bio/libcrypto-lib-bio_meth.o crypto/bio/libcrypto-lib-bss_acpt.o crypto/bio/libcrypto-lib-bss_bio.o crypto/bio/libcrypto-lib-bss_conn.o crypto/bio/libcrypto-lib-bss_dgram.o crypto/bio/libcrypto-lib-bss_fd.o crypto/bio/libcrypto-lib-bss_file.o crypto/bio/libcrypto-lib-bss_log.o crypto/bio/libcrypto-lib-bss_mem.o crypto/bio/libcrypto-lib-bss_null.o crypto/bio/libcrypto-lib-bss_sock.o crypto/bio/libcrypto-lib-core_bio.o crypto/bn/asm/libcrypto-lib-x86_64-gcc.o crypto/bn/libcrypto-lib-bn_add.o crypto/bn/libcrypto-lib-bn_blind.o crypto/bn/libcrypto-lib-bn_const.o crypto/bn/libcrypto-lib-bn_conv.o crypto/bn/libcrypto-lib-bn_ctx.o crypto/bn/libcrypto-lib-bn_depr.o crypto/bn/libcrypto-lib-bn_dh.o crypto/bn/libcrypto-lib-bn_div.o crypto/bn/libcrypto-lib-bn_err.o crypto/bn/libcrypto-lib-bn_exp.o crypto/bn/libcrypto-lib-bn_exp2.o crypto/bn/libcrypto-lib-bn_gcd.o crypto/bn/libcrypto-lib-bn_gf2m.o crypto/bn/libcrypto-lib-bn_intern.o crypto/bn/libcrypto-lib-bn_kron.o crypto/bn/libcrypto-lib-bn_lib.o crypto/bn/libcrypto-lib-bn_mod.o crypto/bn/libcrypto-lib-bn_mont.o crypto/bn/libcrypto-lib-bn_mpi.o crypto/bn/libcrypto-lib-bn_mul.o crypto/bn/libcrypto-lib-bn_nist.o crypto/bn/libcrypto-lib-bn_prime.o crypto/bn/libcrypto-lib-bn_print.o crypto/bn/libcrypto-lib-bn_rand.o crypto/bn/libcrypto-lib-bn_recp.o crypto/bn/libcrypto-lib-bn_rsa_fips186_4.o crypto/bn/libcrypto-lib-bn_shift.o crypto/bn/libcrypto-lib-bn_sqr.o crypto/bn/libcrypto-lib-bn_sqrt.o crypto/bn/libcrypto-lib-bn_srp.o crypto/bn/libcrypto-lib-bn_word.o crypto/bn/libcrypto-lib-bn_x931p.o crypto/bn/libcrypto-lib-rsaz-avx2.o crypto/bn/libcrypto-lib-rsaz-avx512.o crypto/bn/libcrypto-lib-rsaz-x86_64.o crypto/bn/libcrypto-lib-rsaz_exp.o crypto/bn/libcrypto-lib-rsaz_exp_x2.o crypto/bn/libcrypto-lib-x86_64-gf2m.o crypto/bn/libcrypto-lib-x86_64-mont.o crypto/bn/libcrypto-lib-x86_64-mont5.o crypto/buffer/libcrypto-lib-buf_err.o crypto/buffer/libcrypto-lib-buffer.o crypto/camellia/libcrypto-lib-cmll-x86_64.o crypto/camellia/libcrypto-lib-cmll_cfb.o crypto/camellia/libcrypto-lib-cmll_ctr.o crypto/camellia/libcrypto-lib-cmll_ecb.o crypto/camellia/libcrypto-lib-cmll_misc.o crypto/camellia/libcrypto-lib-cmll_ofb.o crypto/cast/libcrypto-lib-c_cfb64.o crypto/cast/libcrypto-lib-c_ecb.o crypto/cast/libcrypto-lib-c_enc.o crypto/cast/libcrypto-lib-c_ofb64.o crypto/cast/libcrypto-lib-c_skey.o crypto/chacha/libcrypto-lib-chacha-x86_64.o crypto/cmac/libcrypto-lib-cmac.o crypto/cmp/libcrypto-lib-cmp_asn.o crypto/cmp/libcrypto-lib-cmp_client.o crypto/cmp/libcrypto-lib-cmp_ctx.o crypto/cmp/libcrypto-lib-cmp_err.o crypto/cmp/libcrypto-lib-cmp_hdr.o crypto/cmp/libcrypto-lib-cmp_http.o crypto/cmp/libcrypto-lib-cmp_msg.o crypto/cmp/libcrypto-lib-cmp_protect.o crypto/cmp/libcrypto-lib-cmp_server.o crypto/cmp/libcrypto-lib-cmp_status.o crypto/cmp/libcrypto-lib-cmp_util.o crypto/cmp/libcrypto-lib-cmp_vfy.o crypto/cms/libcrypto-lib-cms_asn1.o crypto/cms/libcrypto-lib-cms_att.o crypto/cms/libcrypto-lib-cms_cd.o crypto/cms/libcrypto-lib-cms_dd.o crypto/cms/libcrypto-lib-cms_dh.o crypto/cms/libcrypto-lib-cms_ec.o crypto/cms/libcrypto-lib-cms_enc.o crypto/cms/libcrypto-lib-cms_env.o crypto/cms/libcrypto-lib-cms_err.o crypto/cms/libcrypto-lib-cms_ess.o crypto/cms/libcrypto-lib-cms_io.o crypto/cms/libcrypto-lib-cms_kari.o crypto/cms/libcrypto-lib-cms_lib.o crypto/cms/libcrypto-lib-cms_pwri.o crypto/cms/libcrypto-lib-cms_rsa.o crypto/cms/libcrypto-lib-cms_sd.o crypto/cms/libcrypto-lib-cms_smime.o crypto/comp/libcrypto-lib-c_zlib.o crypto/comp/libcrypto-lib-comp_err.o crypto/comp/libcrypto-lib-comp_lib.o crypto/conf/libcrypto-lib-conf_api.o crypto/conf/libcrypto-lib-conf_def.o crypto/conf/libcrypto-lib-conf_err.o crypto/conf/libcrypto-lib-conf_lib.o crypto/conf/libcrypto-lib-conf_mall.o crypto/conf/libcrypto-lib-conf_mod.o crypto/conf/libcrypto-lib-conf_sap.o crypto/conf/libcrypto-lib-conf_ssl.o crypto/crmf/libcrypto-lib-crmf_asn.o crypto/crmf/libcrypto-lib-crmf_err.o crypto/crmf/libcrypto-lib-crmf_lib.o crypto/crmf/libcrypto-lib-crmf_pbm.o crypto/ct/libcrypto-lib-ct_b64.o crypto/ct/libcrypto-lib-ct_err.o crypto/ct/libcrypto-lib-ct_log.o crypto/ct/libcrypto-lib-ct_oct.o crypto/ct/libcrypto-lib-ct_policy.o crypto/ct/libcrypto-lib-ct_prn.o crypto/ct/libcrypto-lib-ct_sct.o crypto/ct/libcrypto-lib-ct_sct_ctx.o crypto/ct/libcrypto-lib-ct_vfy.o crypto/ct/libcrypto-lib-ct_x509v3.o crypto/des/libcrypto-lib-cbc_cksm.o crypto/des/libcrypto-lib-cbc_enc.o crypto/des/libcrypto-lib-cfb64ede.o crypto/des/libcrypto-lib-cfb64enc.o crypto/des/libcrypto-lib-cfb_enc.o crypto/des/libcrypto-lib-des_enc.o crypto/des/libcrypto-lib-ecb3_enc.o crypto/des/libcrypto-lib-ecb_enc.o crypto/des/libcrypto-lib-fcrypt.o crypto/des/libcrypto-lib-fcrypt_b.o crypto/des/libcrypto-lib-ofb64ede.o crypto/des/libcrypto-lib-ofb64enc.o crypto/des/libcrypto-lib-ofb_enc.o crypto/des/libcrypto-lib-pcbc_enc.o crypto/des/libcrypto-lib-qud_cksm.o crypto/des/libcrypto-lib-rand_key.o crypto/des/libcrypto-lib-set_key.o crypto/des/libcrypto-lib-str2key.o crypto/des/libcrypto-lib-xcbc_enc.o crypto/dh/libcrypto-lib-dh_ameth.o crypto/dh/libcrypto-lib-dh_asn1.o crypto/dh/libcrypto-lib-dh_backend.o crypto/dh/libcrypto-lib-dh_check.o crypto/dh/libcrypto-lib-dh_depr.o crypto/dh/libcrypto-lib-dh_err.o crypto/dh/libcrypto-lib-dh_gen.o crypto/dh/libcrypto-lib-dh_group_params.o crypto/dh/libcrypto-lib-dh_kdf.o crypto/dh/libcrypto-lib-dh_key.o crypto/dh/libcrypto-lib-dh_lib.o crypto/dh/libcrypto-lib-dh_meth.o crypto/dh/libcrypto-lib-dh_pmeth.o crypto/dh/libcrypto-lib-dh_prn.o crypto/dh/libcrypto-lib-dh_rfc5114.o crypto/dsa/libcrypto-lib-dsa_ameth.o crypto/dsa/libcrypto-lib-dsa_asn1.o crypto/dsa/libcrypto-lib-dsa_backend.o crypto/dsa/libcrypto-lib-dsa_check.o crypto/dsa/libcrypto-lib-dsa_depr.o crypto/dsa/libcrypto-lib-dsa_err.o crypto/dsa/libcrypto-lib-dsa_gen.o crypto/dsa/libcrypto-lib-dsa_key.o crypto/dsa/libcrypto-lib-dsa_lib.o crypto/dsa/libcrypto-lib-dsa_meth.o crypto/dsa/libcrypto-lib-dsa_ossl.o crypto/dsa/libcrypto-lib-dsa_pmeth.o crypto/dsa/libcrypto-lib-dsa_prn.o crypto/dsa/libcrypto-lib-dsa_sign.o crypto/dsa/libcrypto-lib-dsa_vrf.o crypto/dso/libcrypto-lib-dso_dl.o crypto/dso/libcrypto-lib-dso_dlfcn.o crypto/dso/libcrypto-lib-dso_err.o crypto/dso/libcrypto-lib-dso_lib.o crypto/dso/libcrypto-lib-dso_openssl.o crypto/dso/libcrypto-lib-dso_vms.o crypto/dso/libcrypto-lib-dso_win32.o crypto/ec/curve448/arch_32/libcrypto-lib-f_impl32.o crypto/ec/curve448/arch_64/libcrypto-lib-f_impl64.o crypto/ec/curve448/libcrypto-lib-curve448.o crypto/ec/curve448/libcrypto-lib-curve448_tables.o crypto/ec/curve448/libcrypto-lib-eddsa.o crypto/ec/curve448/libcrypto-lib-f_generic.o crypto/ec/curve448/libcrypto-lib-scalar.o crypto/ec/libcrypto-lib-curve25519.o crypto/ec/libcrypto-lib-ec2_oct.o crypto/ec/libcrypto-lib-ec2_smpl.o crypto/ec/libcrypto-lib-ec_ameth.o crypto/ec/libcrypto-lib-ec_asn1.o crypto/ec/libcrypto-lib-ec_backend.o crypto/ec/libcrypto-lib-ec_check.o crypto/ec/libcrypto-lib-ec_curve.o crypto/ec/libcrypto-lib-ec_cvt.o crypto/ec/libcrypto-lib-ec_deprecated.o crypto/ec/libcrypto-lib-ec_err.o crypto/ec/libcrypto-lib-ec_key.o crypto/ec/libcrypto-lib-ec_kmeth.o crypto/ec/libcrypto-lib-ec_lib.o crypto/ec/libcrypto-lib-ec_mult.o crypto/ec/libcrypto-lib-ec_oct.o crypto/ec/libcrypto-lib-ec_pmeth.o crypto/ec/libcrypto-lib-ec_print.o crypto/ec/libcrypto-lib-ecdh_kdf.o crypto/ec/libcrypto-lib-ecdh_ossl.o crypto/ec/libcrypto-lib-ecdsa_ossl.o crypto/ec/libcrypto-lib-ecdsa_sign.o crypto/ec/libcrypto-lib-ecdsa_vrf.o crypto/ec/libcrypto-lib-eck_prn.o crypto/ec/libcrypto-lib-ecp_mont.o crypto/ec/libcrypto-lib-ecp_nist.o crypto/ec/libcrypto-lib-ecp_nistp224.o crypto/ec/libcrypto-lib-ecp_nistp256.o crypto/ec/libcrypto-lib-ecp_nistp521.o crypto/ec/libcrypto-lib-ecp_nistputil.o crypto/ec/libcrypto-lib-ecp_nistz256-x86_64.o crypto/ec/libcrypto-lib-ecp_nistz256.o crypto/ec/libcrypto-lib-ecp_oct.o crypto/ec/libcrypto-lib-ecp_smpl.o crypto/ec/libcrypto-lib-ecx_backend.o crypto/ec/libcrypto-lib-ecx_key.o crypto/ec/libcrypto-lib-ecx_meth.o crypto/ec/libcrypto-lib-x25519-x86_64.o crypto/encode_decode/libcrypto-lib-decoder_err.o crypto/encode_decode/libcrypto-lib-decoder_lib.o crypto/encode_decode/libcrypto-lib-decoder_meth.o crypto/encode_decode/libcrypto-lib-decoder_pkey.o crypto/encode_decode/libcrypto-lib-encoder_err.o crypto/encode_decode/libcrypto-lib-encoder_lib.o crypto/encode_decode/libcrypto-lib-encoder_meth.o crypto/encode_decode/libcrypto-lib-encoder_pkey.o crypto/engine/libcrypto-lib-eng_all.o crypto/engine/libcrypto-lib-eng_cnf.o crypto/engine/libcrypto-lib-eng_ctrl.o crypto/engine/libcrypto-lib-eng_dyn.o crypto/engine/libcrypto-lib-eng_err.o crypto/engine/libcrypto-lib-eng_fat.o crypto/engine/libcrypto-lib-eng_init.o crypto/engine/libcrypto-lib-eng_lib.o crypto/engine/libcrypto-lib-eng_list.o crypto/engine/libcrypto-lib-eng_openssl.o crypto/engine/libcrypto-lib-eng_pkey.o crypto/engine/libcrypto-lib-eng_rdrand.o crypto/engine/libcrypto-lib-eng_table.o crypto/engine/libcrypto-lib-tb_asnmth.o crypto/engine/libcrypto-lib-tb_cipher.o crypto/engine/libcrypto-lib-tb_dh.o crypto/engine/libcrypto-lib-tb_digest.o crypto/engine/libcrypto-lib-tb_dsa.o crypto/engine/libcrypto-lib-tb_eckey.o crypto/engine/libcrypto-lib-tb_pkmeth.o crypto/engine/libcrypto-lib-tb_rand.o crypto/engine/libcrypto-lib-tb_rsa.o crypto/err/libcrypto-lib-err.o crypto/err/libcrypto-lib-err_all.o crypto/err/libcrypto-lib-err_all_legacy.o crypto/err/libcrypto-lib-err_blocks.o crypto/err/libcrypto-lib-err_prn.o crypto/ess/libcrypto-lib-ess_asn1.o crypto/ess/libcrypto-lib-ess_err.o crypto/ess/libcrypto-lib-ess_lib.o crypto/evp/libcrypto-lib-asymcipher.o crypto/evp/libcrypto-lib-bio_b64.o crypto/evp/libcrypto-lib-bio_enc.o crypto/evp/libcrypto-lib-bio_md.o crypto/evp/libcrypto-lib-bio_ok.o crypto/evp/libcrypto-lib-c_allc.o crypto/evp/libcrypto-lib-c_alld.o crypto/evp/libcrypto-lib-cmeth_lib.o crypto/evp/libcrypto-lib-ctrl_params_translate.o crypto/evp/libcrypto-lib-dh_ctrl.o crypto/evp/libcrypto-lib-dh_support.o crypto/evp/libcrypto-lib-digest.o crypto/evp/libcrypto-lib-dsa_ctrl.o crypto/evp/libcrypto-lib-e_aes.o crypto/evp/libcrypto-lib-e_aes_cbc_hmac_sha1.o crypto/evp/libcrypto-lib-e_aes_cbc_hmac_sha256.o crypto/evp/libcrypto-lib-e_aria.o crypto/evp/libcrypto-lib-e_bf.o crypto/evp/libcrypto-lib-e_camellia.o crypto/evp/libcrypto-lib-e_cast.o crypto/evp/libcrypto-lib-e_chacha20_poly1305.o crypto/evp/libcrypto-lib-e_des.o crypto/evp/libcrypto-lib-e_des3.o crypto/evp/libcrypto-lib-e_idea.o crypto/evp/libcrypto-lib-e_null.o crypto/evp/libcrypto-lib-e_old.o crypto/evp/libcrypto-lib-e_rc2.o crypto/evp/libcrypto-lib-e_rc4.o crypto/evp/libcrypto-lib-e_rc4_hmac_md5.o crypto/evp/libcrypto-lib-e_rc5.o crypto/evp/libcrypto-lib-e_seed.o crypto/evp/libcrypto-lib-e_sm4.o crypto/evp/libcrypto-lib-e_xcbc_d.o crypto/evp/libcrypto-lib-ec_ctrl.o crypto/evp/libcrypto-lib-ec_support.o crypto/evp/libcrypto-lib-encode.o crypto/evp/libcrypto-lib-evp_cnf.o crypto/evp/libcrypto-lib-evp_enc.o crypto/evp/libcrypto-lib-evp_err.o crypto/evp/libcrypto-lib-evp_fetch.o crypto/evp/libcrypto-lib-evp_key.o crypto/evp/libcrypto-lib-evp_lib.o crypto/evp/libcrypto-lib-evp_pbe.o crypto/evp/libcrypto-lib-evp_pkey.o crypto/evp/libcrypto-lib-evp_rand.o crypto/evp/libcrypto-lib-evp_utils.o crypto/evp/libcrypto-lib-exchange.o crypto/evp/libcrypto-lib-kdf_lib.o crypto/evp/libcrypto-lib-kdf_meth.o crypto/evp/libcrypto-lib-kem.o crypto/evp/libcrypto-lib-keymgmt_lib.o crypto/evp/libcrypto-lib-keymgmt_meth.o crypto/evp/libcrypto-lib-legacy_blake2.o crypto/evp/libcrypto-lib-legacy_md2.o crypto/evp/libcrypto-lib-legacy_md4.o crypto/evp/libcrypto-lib-legacy_md5.o crypto/evp/libcrypto-lib-legacy_md5_sha1.o crypto/evp/libcrypto-lib-legacy_mdc2.o crypto/evp/libcrypto-lib-legacy_ripemd.o crypto/evp/libcrypto-lib-legacy_sha.o crypto/evp/libcrypto-lib-legacy_wp.o crypto/evp/libcrypto-lib-m_null.o crypto/evp/libcrypto-lib-m_sigver.o crypto/evp/libcrypto-lib-mac_lib.o crypto/evp/libcrypto-lib-mac_meth.o crypto/evp/libcrypto-lib-names.o crypto/evp/libcrypto-lib-p5_crpt.o crypto/evp/libcrypto-lib-p5_crpt2.o crypto/evp/libcrypto-lib-p_dec.o crypto/evp/libcrypto-lib-p_enc.o crypto/evp/libcrypto-lib-p_legacy.o crypto/evp/libcrypto-lib-p_lib.o crypto/evp/libcrypto-lib-p_open.o crypto/evp/libcrypto-lib-p_seal.o crypto/evp/libcrypto-lib-p_sign.o crypto/evp/libcrypto-lib-p_verify.o crypto/evp/libcrypto-lib-pbe_scrypt.o crypto/evp/libcrypto-lib-pmeth_check.o crypto/evp/libcrypto-lib-pmeth_gn.o crypto/evp/libcrypto-lib-pmeth_lib.o crypto/evp/libcrypto-lib-signature.o crypto/ffc/libcrypto-lib-ffc_backend.o crypto/ffc/libcrypto-lib-ffc_dh.o crypto/ffc/libcrypto-lib-ffc_key_generate.o crypto/ffc/libcrypto-lib-ffc_key_validate.o crypto/ffc/libcrypto-lib-ffc_params.o crypto/ffc/libcrypto-lib-ffc_params_generate.o crypto/ffc/libcrypto-lib-ffc_params_validate.o crypto/hmac/libcrypto-lib-hmac.o crypto/http/libcrypto-lib-http_client.o crypto/http/libcrypto-lib-http_err.o crypto/http/libcrypto-lib-http_lib.o crypto/idea/libcrypto-lib-i_cbc.o crypto/idea/libcrypto-lib-i_cfb64.o crypto/idea/libcrypto-lib-i_ecb.o crypto/idea/libcrypto-lib-i_ofb64.o crypto/idea/libcrypto-lib-i_skey.o crypto/kdf/libcrypto-lib-kdf_err.o crypto/lhash/libcrypto-lib-lh_stats.o crypto/lhash/libcrypto-lib-lhash.o crypto/libcrypto-lib-asn1_dsa.o crypto/libcrypto-lib-bsearch.o crypto/libcrypto-lib-context.o crypto/libcrypto-lib-core_algorithm.o crypto/libcrypto-lib-core_fetch.o crypto/libcrypto-lib-core_namemap.o crypto/libcrypto-lib-cpt_err.o crypto/libcrypto-lib-cpuid.o crypto/libcrypto-lib-cryptlib.o crypto/libcrypto-lib-ctype.o crypto/libcrypto-lib-cversion.o crypto/libcrypto-lib-der_writer.o crypto/libcrypto-lib-ebcdic.o crypto/libcrypto-lib-ex_data.o crypto/libcrypto-lib-getenv.o crypto/libcrypto-lib-info.o crypto/libcrypto-lib-init.o crypto/libcrypto-lib-initthread.o crypto/libcrypto-lib-mem.o crypto/libcrypto-lib-mem_sec.o crypto/libcrypto-lib-o_dir.o crypto/libcrypto-lib-o_fopen.o crypto/libcrypto-lib-o_init.o crypto/libcrypto-lib-o_str.o crypto/libcrypto-lib-o_time.o crypto/libcrypto-lib-packet.o crypto/libcrypto-lib-param_build.o crypto/libcrypto-lib-param_build_set.o crypto/libcrypto-lib-params.o crypto/libcrypto-lib-params_dup.o crypto/libcrypto-lib-params_from_text.o crypto/libcrypto-lib-passphrase.o crypto/libcrypto-lib-provider.o crypto/libcrypto-lib-provider_conf.o crypto/libcrypto-lib-provider_core.o crypto/libcrypto-lib-provider_predefined.o
ar qc libcrypto.a crypto/libcrypto-lib-punycode.o crypto/libcrypto-lib-self_test_core.o crypto/libcrypto-lib-sparse_array.o crypto/libcrypto-lib-threads_lib.o crypto/libcrypto-lib-threads_none.o crypto/libcrypto-lib-threads_pthread.o crypto/libcrypto-lib-threads_win.o crypto/libcrypto-lib-trace.o crypto/libcrypto-lib-uid.o crypto/libcrypto-lib-x86_64cpuid.o crypto/md2/libcrypto-lib-md2_dgst.o crypto/md2/libcrypto-lib-md2_one.o crypto/md4/libcrypto-lib-md4_dgst.o crypto/md4/libcrypto-lib-md4_one.o crypto/md5/libcrypto-lib-md5-x86_64.o crypto/md5/libcrypto-lib-md5_dgst.o crypto/md5/libcrypto-lib-md5_one.o crypto/md5/libcrypto-lib-md5_sha1.o crypto/mdc2/libcrypto-lib-mdc2_one.o crypto/mdc2/libcrypto-lib-mdc2dgst.o crypto/modes/libcrypto-lib-aesni-gcm-x86_64.o crypto/modes/libcrypto-lib-cbc128.o crypto/modes/libcrypto-lib-ccm128.o crypto/modes/libcrypto-lib-cfb128.o crypto/modes/libcrypto-lib-ctr128.o crypto/modes/libcrypto-lib-cts128.o crypto/modes/libcrypto-lib-gcm128.o crypto/modes/libcrypto-lib-ghash-x86_64.o crypto/modes/libcrypto-lib-ocb128.o crypto/modes/libcrypto-lib-ofb128.o crypto/modes/libcrypto-lib-siv128.o crypto/modes/libcrypto-lib-wrap128.o crypto/modes/libcrypto-lib-xts128.o crypto/objects/libcrypto-lib-o_names.o crypto/objects/libcrypto-lib-obj_dat.o crypto/objects/libcrypto-lib-obj_err.o crypto/objects/libcrypto-lib-obj_lib.o crypto/objects/libcrypto-lib-obj_xref.o crypto/ocsp/libcrypto-lib-ocsp_asn.o crypto/ocsp/libcrypto-lib-ocsp_cl.o crypto/ocsp/libcrypto-lib-ocsp_err.o crypto/ocsp/libcrypto-lib-ocsp_ext.o crypto/ocsp/libcrypto-lib-ocsp_http.o crypto/ocsp/libcrypto-lib-ocsp_lib.o crypto/ocsp/libcrypto-lib-ocsp_prn.o crypto/ocsp/libcrypto-lib-ocsp_srv.o crypto/ocsp/libcrypto-lib-ocsp_vfy.o crypto/ocsp/libcrypto-lib-v3_ocsp.o crypto/pem/libcrypto-lib-pem_all.o crypto/pem/libcrypto-lib-pem_err.o crypto/pem/libcrypto-lib-pem_info.o crypto/pem/libcrypto-lib-pem_lib.o crypto/pem/libcrypto-lib-pem_oth.o crypto/pem/libcrypto-lib-pem_pk8.o crypto/pem/libcrypto-lib-pem_pkey.o crypto/pem/libcrypto-lib-pem_sign.o crypto/pem/libcrypto-lib-pem_x509.o crypto/pem/libcrypto-lib-pem_xaux.o crypto/pem/libcrypto-lib-pvkfmt.o crypto/pkcs12/libcrypto-lib-p12_add.o crypto/pkcs12/libcrypto-lib-p12_asn.o crypto/pkcs12/libcrypto-lib-p12_attr.o crypto/pkcs12/libcrypto-lib-p12_crpt.o crypto/pkcs12/libcrypto-lib-p12_crt.o crypto/pkcs12/libcrypto-lib-p12_decr.o crypto/pkcs12/libcrypto-lib-p12_init.o crypto/pkcs12/libcrypto-lib-p12_key.o crypto/pkcs12/libcrypto-lib-p12_kiss.o crypto/pkcs12/libcrypto-lib-p12_mutl.o crypto/pkcs12/libcrypto-lib-p12_npas.o crypto/pkcs12/libcrypto-lib-p12_p8d.o crypto/pkcs12/libcrypto-lib-p12_p8e.o crypto/pkcs12/libcrypto-lib-p12_sbag.o crypto/pkcs12/libcrypto-lib-p12_utl.o crypto/pkcs12/libcrypto-lib-pk12err.o crypto/pkcs7/libcrypto-lib-bio_pk7.o crypto/pkcs7/libcrypto-lib-pk7_asn1.o crypto/pkcs7/libcrypto-lib-pk7_attr.o crypto/pkcs7/libcrypto-lib-pk7_doit.o crypto/pkcs7/libcrypto-lib-pk7_lib.o crypto/pkcs7/libcrypto-lib-pk7_mime.o crypto/pkcs7/libcrypto-lib-pk7_smime.o crypto/pkcs7/libcrypto-lib-pkcs7err.o crypto/poly1305/libcrypto-lib-poly1305-x86_64.o crypto/poly1305/libcrypto-lib-poly1305.o crypto/property/libcrypto-lib-defn_cache.o crypto/property/libcrypto-lib-property.o crypto/property/libcrypto-lib-property_err.o crypto/property/libcrypto-lib-property_parse.o crypto/property/libcrypto-lib-property_string.o crypto/rand/libcrypto-lib-prov_seed.o crypto/rand/libcrypto-lib-rand_deprecated.o crypto/rand/libcrypto-lib-rand_err.o crypto/rand/libcrypto-lib-rand_lib.o crypto/rand/libcrypto-lib-rand_meth.o crypto/rand/libcrypto-lib-rand_pool.o crypto/rand/libcrypto-lib-randfile.o crypto/rc2/libcrypto-lib-rc2_cbc.o crypto/rc2/libcrypto-lib-rc2_ecb.o crypto/rc2/libcrypto-lib-rc2_skey.o crypto/rc2/libcrypto-lib-rc2cfb64.o crypto/rc2/libcrypto-lib-rc2ofb64.o crypto/rc4/libcrypto-lib-rc4-md5-x86_64.o crypto/rc4/libcrypto-lib-rc4-x86_64.o crypto/ripemd/libcrypto-lib-rmd_dgst.o crypto/ripemd/libcrypto-lib-rmd_one.o crypto/rsa/libcrypto-lib-rsa_ameth.o crypto/rsa/libcrypto-lib-rsa_asn1.o crypto/rsa/libcrypto-lib-rsa_backend.o crypto/rsa/libcrypto-lib-rsa_chk.o crypto/rsa/libcrypto-lib-rsa_crpt.o crypto/rsa/libcrypto-lib-rsa_depr.o crypto/rsa/libcrypto-lib-rsa_err.o crypto/rsa/libcrypto-lib-rsa_gen.o crypto/rsa/libcrypto-lib-rsa_lib.o crypto/rsa/libcrypto-lib-rsa_meth.o crypto/rsa/libcrypto-lib-rsa_mp.o crypto/rsa/libcrypto-lib-rsa_mp_names.o crypto/rsa/libcrypto-lib-rsa_none.o crypto/rsa/libcrypto-lib-rsa_oaep.o crypto/rsa/libcrypto-lib-rsa_ossl.o crypto/rsa/libcrypto-lib-rsa_pk1.o crypto/rsa/libcrypto-lib-rsa_pmeth.o crypto/rsa/libcrypto-lib-rsa_prn.o crypto/rsa/libcrypto-lib-rsa_pss.o crypto/rsa/libcrypto-lib-rsa_saos.o crypto/rsa/libcrypto-lib-rsa_schemes.o crypto/rsa/libcrypto-lib-rsa_sign.o crypto/rsa/libcrypto-lib-rsa_sp800_56b_check.o crypto/rsa/libcrypto-lib-rsa_sp800_56b_gen.o crypto/rsa/libcrypto-lib-rsa_x931.o crypto/rsa/libcrypto-lib-rsa_x931g.o crypto/seed/libcrypto-lib-seed.o crypto/seed/libcrypto-lib-seed_cbc.o crypto/seed/libcrypto-lib-seed_cfb.o crypto/seed/libcrypto-lib-seed_ecb.o crypto/seed/libcrypto-lib-seed_ofb.o crypto/sha/libcrypto-lib-keccak1600-x86_64.o crypto/sha/libcrypto-lib-sha1-mb-x86_64.o crypto/sha/libcrypto-lib-sha1-x86_64.o crypto/sha/libcrypto-lib-sha1_one.o crypto/sha/libcrypto-lib-sha1dgst.o crypto/sha/libcrypto-lib-sha256-mb-x86_64.o crypto/sha/libcrypto-lib-sha256-x86_64.o crypto/sha/libcrypto-lib-sha256.o crypto/sha/libcrypto-lib-sha3.o crypto/sha/libcrypto-lib-sha512-x86_64.o crypto/sha/libcrypto-lib-sha512.o crypto/siphash/libcrypto-lib-siphash.o crypto/sm2/libcrypto-lib-sm2_crypt.o crypto/sm2/libcrypto-lib-sm2_err.o crypto/sm2/libcrypto-lib-sm2_key.o crypto/sm2/libcrypto-lib-sm2_sign.o crypto/sm3/libcrypto-lib-legacy_sm3.o crypto/sm3/libcrypto-lib-sm3.o crypto/sm4/libcrypto-lib-sm4.o crypto/srp/libcrypto-lib-srp_lib.o crypto/srp/libcrypto-lib-srp_vfy.o crypto/stack/libcrypto-lib-stack.o crypto/store/libcrypto-lib-store_err.o crypto/store/libcrypto-lib-store_init.o crypto/store/libcrypto-lib-store_lib.o crypto/store/libcrypto-lib-store_meth.o crypto/store/libcrypto-lib-store_register.o crypto/store/libcrypto-lib-store_result.o crypto/store/libcrypto-lib-store_strings.o crypto/ts/libcrypto-lib-ts_asn1.o crypto/ts/libcrypto-lib-ts_conf.o crypto/ts/libcrypto-lib-ts_err.o crypto/ts/libcrypto-lib-ts_lib.o crypto/ts/libcrypto-lib-ts_req_print.o crypto/ts/libcrypto-lib-ts_req_utils.o crypto/ts/libcrypto-lib-ts_rsp_print.o crypto/ts/libcrypto-lib-ts_rsp_sign.o crypto/ts/libcrypto-lib-ts_rsp_utils.o crypto/ts/libcrypto-lib-ts_rsp_verify.o crypto/ts/libcrypto-lib-ts_verify_ctx.o crypto/txt_db/libcrypto-lib-txt_db.o crypto/ui/libcrypto-lib-ui_err.o crypto/ui/libcrypto-lib-ui_lib.o crypto/ui/libcrypto-lib-ui_null.o crypto/ui/libcrypto-lib-ui_openssl.o crypto/ui/libcrypto-lib-ui_util.o crypto/whrlpool/libcrypto-lib-wp-x86_64.o crypto/whrlpool/libcrypto-lib-wp_dgst.o crypto/x509/libcrypto-lib-by_dir.o crypto/x509/libcrypto-lib-by_file.o crypto/x509/libcrypto-lib-by_store.o crypto/x509/libcrypto-lib-pcy_cache.o crypto/x509/libcrypto-lib-pcy_data.o crypto/x509/libcrypto-lib-pcy_lib.o crypto/x509/libcrypto-lib-pcy_map.o crypto/x509/libcrypto-lib-pcy_node.o crypto/x509/libcrypto-lib-pcy_tree.o crypto/x509/libcrypto-lib-t_crl.o crypto/x509/libcrypto-lib-t_req.o crypto/x509/libcrypto-lib-t_x509.o crypto/x509/libcrypto-lib-v3_addr.o crypto/x509/libcrypto-lib-v3_admis.o crypto/x509/libcrypto-lib-v3_akeya.o crypto/x509/libcrypto-lib-v3_akid.o crypto/x509/libcrypto-lib-v3_asid.o crypto/x509/libcrypto-lib-v3_bcons.o crypto/x509/libcrypto-lib-v3_bitst.o crypto/x509/libcrypto-lib-v3_conf.o crypto/x509/libcrypto-lib-v3_cpols.o crypto/x509/libcrypto-lib-v3_crld.o crypto/x509/libcrypto-lib-v3_enum.o crypto/x509/libcrypto-lib-v3_extku.o crypto/x509/libcrypto-lib-v3_genn.o crypto/x509/libcrypto-lib-v3_ia5.o crypto/x509/libcrypto-lib-v3_info.o crypto/x509/libcrypto-lib-v3_int.o crypto/x509/libcrypto-lib-v3_ist.o crypto/x509/libcrypto-lib-v3_lib.o crypto/x509/libcrypto-lib-v3_ncons.o crypto/x509/libcrypto-lib-v3_pci.o crypto/x509/libcrypto-lib-v3_pcia.o crypto/x509/libcrypto-lib-v3_pcons.o crypto/x509/libcrypto-lib-v3_pku.o crypto/x509/libcrypto-lib-v3_pmaps.o crypto/x509/libcrypto-lib-v3_prn.o crypto/x509/libcrypto-lib-v3_purp.o crypto/x509/libcrypto-lib-v3_san.o crypto/x509/libcrypto-lib-v3_skid.o crypto/x509/libcrypto-lib-v3_sxnet.o crypto/x509/libcrypto-lib-v3_tlsf.o crypto/x509/libcrypto-lib-v3_utf8.o crypto/x509/libcrypto-lib-v3_utl.o crypto/x509/libcrypto-lib-v3err.o crypto/x509/libcrypto-lib-x509_att.o crypto/x509/libcrypto-lib-x509_cmp.o crypto/x509/libcrypto-lib-x509_d2.o crypto/x509/libcrypto-lib-x509_def.o crypto/x509/libcrypto-lib-x509_err.o crypto/x509/libcrypto-lib-x509_ext.o crypto/x509/libcrypto-lib-x509_lu.o crypto/x509/libcrypto-lib-x509_meth.o crypto/x509/libcrypto-lib-x509_obj.o crypto/x509/libcrypto-lib-x509_r2x.o crypto/x509/libcrypto-lib-x509_req.o crypto/x509/libcrypto-lib-x509_set.o crypto/x509/libcrypto-lib-x509_trs.o crypto/x509/libcrypto-lib-x509_txt.o crypto/x509/libcrypto-lib-x509_v3.o crypto/x509/libcrypto-lib-x509_vfy.o crypto/x509/libcrypto-lib-x509_vpm.o crypto/x509/libcrypto-lib-x509cset.o crypto/x509/libcrypto-lib-x509name.o crypto/x509/libcrypto-lib-x509rset.o crypto/x509/libcrypto-lib-x509spki.o crypto/x509/libcrypto-lib-x509type.o crypto/x509/libcrypto-lib-x_all.o crypto/x509/libcrypto-lib-x_attrib.o crypto/x509/libcrypto-lib-x_crl.o crypto/x509/libcrypto-lib-x_exten.o crypto/x509/libcrypto-lib-x_name.o crypto/x509/libcrypto-lib-x_pubkey.o crypto/x509/libcrypto-lib-x_req.o crypto/x509/libcrypto-lib-x_x509.o crypto/x509/libcrypto-lib-x_x509a.o providers/libcrypto-lib-baseprov.o providers/libcrypto-lib-defltprov.o providers/libcrypto-lib-nullprov.o providers/libcrypto-lib-prov_running.o crypto/md5/libimplementations-lib-md5-x86_64.o crypto/md5/libimplementations-lib-md5_dgst.o crypto/md5/libimplementations-lib-md5_one.o crypto/md5/libimplementations-lib-md5_sha1.o providers/implementations/asymciphers/libimplementations-lib-rsa_enc.o providers/implementations/asymciphers/libimplementations-lib-sm2_enc.o providers/implementations/ciphers/libimplementations-lib-cipher_aes.o providers/implementations/ciphers/libimplementations-lib-cipher_aes_cbc_hmac_sha.o providers/implementations/ciphers/libimplementations-lib-cipher_aes_cbc_hmac_sha1_hw.o providers/implementations/ciphers/libimplementations-lib-cipher_aes_cbc_hmac_sha256_hw.o providers/implementations/ciphers/libimplementations-lib-cipher_aes_ccm.o providers/implementations/ciphers/libimplementations-lib-cipher_aes_ccm_hw.o providers/implementations/ciphers/libimplementations-lib-cipher_aes_cts.o providers/implementations/ciphers/libimplementations-lib-cipher_aes_gcm.o providers/implementations/ciphers/libimplementations-lib-cipher_aes_gcm_hw.o providers/implementations/ciphers/libimplementations-lib-cipher_aes_hw.o providers/implementations/ciphers/libimplementations-lib-cipher_aes_ocb.o providers/implementations/ciphers/libimplementations-lib-cipher_aes_ocb_hw.o providers/implementations/ciphers/libimplementations-lib-cipher_aes_siv.o providers/implementations/ciphers/libimplementations-lib-cipher_aes_siv_hw.o providers/implementations/ciphers/libimplementations-lib-cipher_aes_wrp.o providers/implementations/ciphers/libimplementations-lib-cipher_aes_xts.o providers/implementations/ciphers/libimplementations-lib-cipher_aes_xts_hw.o providers/implementations/ciphers/libimplementations-lib-cipher_aria.o providers/implementations/ciphers/libimplementations-lib-cipher_aria_ccm.o providers/implementations/ciphers/libimplementations-lib-cipher_aria_ccm_hw.o providers/implementations/ciphers/libimplementations-lib-cipher_aria_gcm.o providers/implementations/ciphers/libimplementations-lib-cipher_aria_gcm_hw.o providers/implementations/ciphers/libimplementations-lib-cipher_aria_hw.o providers/implementations/ciphers/libimplementations-lib-cipher_camellia.o providers/implementations/ciphers/libimplementations-lib-cipher_camellia_hw.o providers/implementations/ciphers/libimplementations-lib-cipher_chacha20.o providers/implementations/ciphers/libimplementations-lib-cipher_chacha20_hw.o providers/implementations/ciphers/libimplementations-lib-cipher_chacha20_poly1305.o providers/implementations/ciphers/libimplementations-lib-cipher_chacha20_poly1305_hw.o providers/implementations/ciphers/libimplementations-lib-cipher_null.o providers/implementations/ciphers/libimplementations-lib-cipher_sm4.o providers/implementations/ciphers/libimplementations-lib-cipher_sm4_hw.o providers/implementations/ciphers/libimplementations-lib-cipher_tdes.o providers/implementations/ciphers/libimplementations-lib-cipher_tdes_common.o providers/implementations/ciphers/libimplementations-lib-cipher_tdes_default.o providers/implementations/ciphers/libimplementations-lib-cipher_tdes_default_hw.o providers/implementations/ciphers/libimplementations-lib-cipher_tdes_hw.o providers/implementations/ciphers/libimplementations-lib-cipher_tdes_wrap.o providers/implementations/ciphers/libimplementations-lib-cipher_tdes_wrap_hw.o providers/implementations/digests/libimplementations-lib-blake2_prov.o providers/implementations/digests/libimplementations-lib-blake2b_prov.o providers/implementations/digests/libimplementations-lib-blake2s_prov.o providers/implementations/digests/libimplementations-lib-md5_prov.o providers/implementations/digests/libimplementations-lib-md5_sha1_prov.o providers/implementations/digests/libimplementations-lib-sha2_prov.o providers/implementations/digests/libimplementations-lib-sha3_prov.o providers/implementations/digests/libimplementations-lib-sm3_prov.o providers/implementations/encode_decode/libimplementations-lib-decode_der2key.o providers/implementations/encode_decode/libimplementations-lib-decode_msblob2key.o providers/implementations/encode_decode/libimplementations-lib-decode_pem2der.o providers/implementations/encode_decode/libimplementations-lib-decode_pvk2key.o providers/implementations/encode_decode/libimplementations-lib-encode_key2any.o providers/implementations/encode_decode/libimplementations-lib-encode_key2blob.o providers/implementations/encode_decode/libimplementations-lib-encode_key2ms.o providers/implementations/encode_decode/libimplementations-lib-encode_key2text.o providers/implementations/encode_decode/libimplementations-lib-endecoder_common.o providers/implementations/exchange/libimplementations-lib-dh_exch.o providers/implementations/exchange/libimplementations-lib-ecdh_exch.o providers/implementations/exchange/libimplementations-lib-ecx_exch.o providers/implementations/exchange/libimplementations-lib-kdf_exch.o providers/implementations/kdfs/libimplementations-lib-hkdf.o providers/implementations/kdfs/libimplementations-lib-kbkdf.o providers/implementations/kdfs/libimplementations-lib-krb5kdf.o providers/implementations/kdfs/libimplementations-lib-pbkdf2.o providers/implementations/kdfs/libimplementations-lib-pkcs12kdf.o providers/implementations/kdfs/libimplementations-lib-scrypt.o providers/implementations/kdfs/libimplementations-lib-sshkdf.o providers/implementations/kdfs/libimplementations-lib-sskdf.o providers/implementations/kdfs/libimplementations-lib-tls1_prf.o providers/implementations/kdfs/libimplementations-lib-x942kdf.o providers/implementations/kem/libimplementations-lib-rsa_kem.o providers/implementations/keymgmt/libimplementations-lib-ecx_kmgmt.o providers/implementations/keymgmt/libimplementations-lib-kdf_legacy_kmgmt.o providers/implementations/macs/libimplementations-lib-blake2b_mac.o providers/implementations/macs/libimplementations-lib-blake2s_mac.o providers/implementations/macs/libimplementations-lib-cmac_prov.o providers/implementations/macs/libimplementations-lib-gmac_prov.o providers/implementations/macs/libimplementations-lib-hmac_prov.o providers/implementations/macs/libimplementations-lib-kmac_prov.o providers/implementations/macs/libimplementations-lib-poly1305_prov.o providers/implementations/macs/libimplementations-lib-siphash_prov.o providers/implementations/signature/libimplementations-lib-dsa.o providers/implementations/signature/libimplementations-lib-ecdsa.o providers/implementations/signature/libimplementations-lib-eddsa.o providers/implementations/signature/libimplementations-lib-sm2sig.o providers/implementations/storemgmt/libimplementations-lib-file_store.o providers/implementations/storemgmt/libimplementations-lib-file_store_der2obj.o ssl/libimplementations-lib-s3_cbc.o providers/common/libcommon-lib-provider_ctx.o providers/common/libcommon-lib-provider_err.o providers/implementations/ciphers/libcommon-lib-ciphercommon.o providers/implementations/ciphers/libcommon-lib-ciphercommon_block.o providers/implementations/ciphers/libcommon-lib-ciphercommon_ccm.o providers/implementations/ciphers/libcommon-lib-ciphercommon_ccm_hw.o providers/implementations/ciphers/libcommon-lib-ciphercommon_gcm.o providers/implementations/ciphers/libcommon-lib-ciphercommon_gcm_hw.o providers/implementations/ciphers/libcommon-lib-ciphercommon_hw.o providers/implementations/digests/libcommon-lib-digestcommon.o ssl/record/libcommon-lib-tls_pad.o providers/common/der/libnonfips-lib-der_digests_gen.o providers/common/der/libnonfips-lib-der_dsa_gen.o providers/common/der/libnonfips-lib-der_dsa_key.o providers/common/der/libnonfips-lib-der_dsa_sig.o providers/common/der/libnonfips-lib-der_ec_gen.o providers/common/der/libnonfips-lib-der_ec_key.o providers/common/der/libnonfips-lib-der_ec_sig.o providers/common/der/libnonfips-lib-der_ecx_gen.o providers/common/der/libnonfips-lib-der_ecx_key.o providers/common/der/libnonfips-lib-der_rsa_gen.o providers/common/der/libnonfips-lib-der_rsa_key.o providers/common/der/libnonfips-lib-der_rsa_sig.o providers/common/der/libnonfips-lib-der_sm2_gen.o providers/common/der/libnonfips-lib-der_sm2_key.o providers/common/der/libnonfips-lib-der_sm2_sig.o providers/common/der/libnonfips-lib-der_wrap_gen.o providers/common/libnonfips-lib-bio_prov.o providers/common/libnonfips-lib-capabilities.o providers/common/libnonfips-lib-digest_to_nid.o providers/common/libnonfips-lib-provider_seeding.o providers/common/libnonfips-lib-provider_util.o providers/common/libnonfips-lib-securitycheck.o providers/common/libnonfips-lib-securitycheck_default.o providers/implementations/ciphers/libnonfips-lib-cipher_aes_xts_fips.o providers/implementations/kdfs/libnonfips-lib-pbkdf2_fips.o providers/implementations/keymgmt/libnonfips-lib-dh_kmgmt.o providers/implementations/keymgmt/libnonfips-lib-dsa_kmgmt.o providers/implementations/keymgmt/libnonfips-lib-ec_kmgmt.o providers/implementations/keymgmt/libnonfips-lib-mac_legacy_kmgmt.o providers/implementations/keymgmt/libnonfips-lib-rsa_kmgmt.o providers/implementations/rands/libnonfips-lib-crngt.o providers/implementations/rands/libnonfips-lib-drbg.o providers/implementations/rands/libnonfips-lib-drbg_ctr.o providers/implementations/rands/libnonfips-lib-drbg_hash.o providers/implementations/rands/libnonfips-lib-drbg_hmac.o providers/implementations/rands/libnonfips-lib-seed_src.o providers/implementations/rands/libnonfips-lib-test_rng.o providers/implementations/rands/seeding/libnonfips-lib-rand_cpu_x86.o providers/implementations/rands/seeding/libnonfips-lib-rand_tsc.o providers/implementations/rands/seeding/libnonfips-lib-rand_unix.o providers/implementations/rands/seeding/libnonfips-lib-rand_win.o providers/implementations/signature/libnonfips-lib-mac_legacy.o providers/implementations/signature/libnonfips-lib-rsa.o providers/libnonfips-lib-prov_running.o
ranlib libcrypto.a || echo Never mind.
ccache clang  -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -DAES_ASM -DBSAES_ASM -DECP_NISTZ256_ASM -DFIPS_MODULE -DGHASH_ASM -DKECCAK1600_ASM -DOPENSSL_BN_ASM_GF2m -DOPENSSL_BN_ASM_MONT -DOPENSSL_BN_ASM_MONT5 -DOPENSSL_CPUID_OBJ -DOPENSSL_IA32_SSE2 -DSHA1_ASM -DSHA256_ASM -DSHA512_ASM -DVPAES_ASM -DX25519_ASM -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF crypto/evp/libfips-lib-keymgmt_lib.d.tmp -MT crypto/evp/libfips-lib-keymgmt_lib.o -c -o crypto/evp/libfips-lib-keymgmt_lib.o crypto/evp/keymgmt_lib.c
ccache clang  -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -DAES_ASM -DBSAES_ASM -DECP_NISTZ256_ASM -DFIPS_MODULE -DGHASH_ASM -DKECCAK1600_ASM -DOPENSSL_BN_ASM_GF2m -DOPENSSL_BN_ASM_MONT -DOPENSSL_BN_ASM_MONT5 -DOPENSSL_CPUID_OBJ -DOPENSSL_IA32_SSE2 -DSHA1_ASM -DSHA256_ASM -DSHA512_ASM -DVPAES_ASM -DX25519_ASM -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF crypto/evp/libfips-lib-p_lib.d.tmp -MT crypto/evp/libfips-lib-p_lib.o -c -o crypto/evp/libfips-lib-p_lib.o crypto/evp/p_lib.c
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -DAES_ASM -DBSAES_ASM -DECP_NISTZ256_ASM -DFIPS_MODULE -DGHASH_ASM -DKECCAK1600_ASM -DOPENSSL_BN_ASM_GF2m -DOPENSSL_BN_ASM_MONT -DOPENSSL_BN_ASM_MONT5 -DOPENSSL_CPUID_OBJ -DOPENSSL_IA32_SSE2 -DSHA1_ASM -DSHA256_ASM -DSHA512_ASM -DVPAES_ASM -DX25519_ASM -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/common/der/libfips-lib-der_digests_gen.d.tmp -MT providers/common/der/libfips-lib-der_digests_gen.o -c -o providers/common/der/libfips-lib-der_digests_gen.o providers/common/der/der_digests_gen.c
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -DAES_ASM -DBSAES_ASM -DECP_NISTZ256_ASM -DFIPS_MODULE -DGHASH_ASM -DKECCAK1600_ASM -DOPENSSL_BN_ASM_GF2m -DOPENSSL_BN_ASM_MONT -DOPENSSL_BN_ASM_MONT5 -DOPENSSL_CPUID_OBJ -DOPENSSL_IA32_SSE2 -DSHA1_ASM -DSHA256_ASM -DSHA512_ASM -DVPAES_ASM -DX25519_ASM -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/common/der/libfips-lib-der_dsa_gen.d.tmp -MT providers/common/der/libfips-lib-der_dsa_gen.o -c -o providers/common/der/libfips-lib-der_dsa_gen.o providers/common/der/der_dsa_gen.c
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -DAES_ASM -DBSAES_ASM -DECP_NISTZ256_ASM -DFIPS_MODULE -DGHASH_ASM -DKECCAK1600_ASM -DOPENSSL_BN_ASM_GF2m -DOPENSSL_BN_ASM_MONT -DOPENSSL_BN_ASM_MONT5 -DOPENSSL_CPUID_OBJ -DOPENSSL_IA32_SSE2 -DSHA1_ASM -DSHA256_ASM -DSHA512_ASM -DVPAES_ASM -DX25519_ASM -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/common/der/libfips-lib-der_dsa_key.d.tmp -MT providers/common/der/libfips-lib-der_dsa_key.o -c -o providers/common/der/libfips-lib-der_dsa_key.o providers/common/der/der_dsa_key.c
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -DAES_ASM -DBSAES_ASM -DECP_NISTZ256_ASM -DFIPS_MODULE -DGHASH_ASM -DKECCAK1600_ASM -DOPENSSL_BN_ASM_GF2m -DOPENSSL_BN_ASM_MONT -DOPENSSL_BN_ASM_MONT5 -DOPENSSL_CPUID_OBJ -DOPENSSL_IA32_SSE2 -DSHA1_ASM -DSHA256_ASM -DSHA512_ASM -DVPAES_ASM -DX25519_ASM -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/common/der/libfips-lib-der_dsa_sig.d.tmp -MT providers/common/der/libfips-lib-der_dsa_sig.o -c -o providers/common/der/libfips-lib-der_dsa_sig.o providers/common/der/der_dsa_sig.c
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -DAES_ASM -DBSAES_ASM -DECP_NISTZ256_ASM -DFIPS_MODULE -DGHASH_ASM -DKECCAK1600_ASM -DOPENSSL_BN_ASM_GF2m -DOPENSSL_BN_ASM_MONT -DOPENSSL_BN_ASM_MONT5 -DOPENSSL_CPUID_OBJ -DOPENSSL_IA32_SSE2 -DSHA1_ASM -DSHA256_ASM -DSHA512_ASM -DVPAES_ASM -DX25519_ASM -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/common/der/libfips-lib-der_ec_gen.d.tmp -MT providers/common/der/libfips-lib-der_ec_gen.o -c -o providers/common/der/libfips-lib-der_ec_gen.o providers/common/der/der_ec_gen.c
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -DAES_ASM -DBSAES_ASM -DECP_NISTZ256_ASM -DFIPS_MODULE -DGHASH_ASM -DKECCAK1600_ASM -DOPENSSL_BN_ASM_GF2m -DOPENSSL_BN_ASM_MONT -DOPENSSL_BN_ASM_MONT5 -DOPENSSL_CPUID_OBJ -DOPENSSL_IA32_SSE2 -DSHA1_ASM -DSHA256_ASM -DSHA512_ASM -DVPAES_ASM -DX25519_ASM -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/common/der/libfips-lib-der_ec_key.d.tmp -MT providers/common/der/libfips-lib-der_ec_key.o -c -o providers/common/der/libfips-lib-der_ec_key.o providers/common/der/der_ec_key.c
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -DAES_ASM -DBSAES_ASM -DECP_NISTZ256_ASM -DFIPS_MODULE -DGHASH_ASM -DKECCAK1600_ASM -DOPENSSL_BN_ASM_GF2m -DOPENSSL_BN_ASM_MONT -DOPENSSL_BN_ASM_MONT5 -DOPENSSL_CPUID_OBJ -DOPENSSL_IA32_SSE2 -DSHA1_ASM -DSHA256_ASM -DSHA512_ASM -DVPAES_ASM -DX25519_ASM -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/common/der/libfips-lib-der_ec_sig.d.tmp -MT providers/common/der/libfips-lib-der_ec_sig.o -c -o providers/common/der/libfips-lib-der_ec_sig.o providers/common/der/der_ec_sig.c
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -DAES_ASM -DBSAES_ASM -DECP_NISTZ256_ASM -DFIPS_MODULE -DGHASH_ASM -DKECCAK1600_ASM -DOPENSSL_BN_ASM_GF2m -DOPENSSL_BN_ASM_MONT -DOPENSSL_BN_ASM_MONT5 -DOPENSSL_CPUID_OBJ -DOPENSSL_IA32_SSE2 -DSHA1_ASM -DSHA256_ASM -DSHA512_ASM -DVPAES_ASM -DX25519_ASM -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/common/der/libfips-lib-der_ecx_gen.d.tmp -MT providers/common/der/libfips-lib-der_ecx_gen.o -c -o providers/common/der/libfips-lib-der_ecx_gen.o providers/common/der/der_ecx_gen.c
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -DAES_ASM -DBSAES_ASM -DECP_NISTZ256_ASM -DFIPS_MODULE -DGHASH_ASM -DKECCAK1600_ASM -DOPENSSL_BN_ASM_GF2m -DOPENSSL_BN_ASM_MONT -DOPENSSL_BN_ASM_MONT5 -DOPENSSL_CPUID_OBJ -DOPENSSL_IA32_SSE2 -DSHA1_ASM -DSHA256_ASM -DSHA512_ASM -DVPAES_ASM -DX25519_ASM -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/common/der/libfips-lib-der_ecx_key.d.tmp -MT providers/common/der/libfips-lib-der_ecx_key.o -c -o providers/common/der/libfips-lib-der_ecx_key.o providers/common/der/der_ecx_key.c
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -DAES_ASM -DBSAES_ASM -DECP_NISTZ256_ASM -DFIPS_MODULE -DGHASH_ASM -DKECCAK1600_ASM -DOPENSSL_BN_ASM_GF2m -DOPENSSL_BN_ASM_MONT -DOPENSSL_BN_ASM_MONT5 -DOPENSSL_CPUID_OBJ -DOPENSSL_IA32_SSE2 -DSHA1_ASM -DSHA256_ASM -DSHA512_ASM -DVPAES_ASM -DX25519_ASM -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/common/der/libfips-lib-der_rsa_gen.d.tmp -MT providers/common/der/libfips-lib-der_rsa_gen.o -c -o providers/common/der/libfips-lib-der_rsa_gen.o providers/common/der/der_rsa_gen.c
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -DAES_ASM -DBSAES_ASM -DECP_NISTZ256_ASM -DFIPS_MODULE -DGHASH_ASM -DKECCAK1600_ASM -DOPENSSL_BN_ASM_GF2m -DOPENSSL_BN_ASM_MONT -DOPENSSL_BN_ASM_MONT5 -DOPENSSL_CPUID_OBJ -DOPENSSL_IA32_SSE2 -DSHA1_ASM -DSHA256_ASM -DSHA512_ASM -DVPAES_ASM -DX25519_ASM -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/common/der/libfips-lib-der_rsa_key.d.tmp -MT providers/common/der/libfips-lib-der_rsa_key.o -c -o providers/common/der/libfips-lib-der_rsa_key.o providers/common/der/der_rsa_key.c
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -DAES_ASM -DBSAES_ASM -DECP_NISTZ256_ASM -DFIPS_MODULE -DGHASH_ASM -DKECCAK1600_ASM -DOPENSSL_BN_ASM_GF2m -DOPENSSL_BN_ASM_MONT -DOPENSSL_BN_ASM_MONT5 -DOPENSSL_CPUID_OBJ -DOPENSSL_IA32_SSE2 -DSHA1_ASM -DSHA256_ASM -DSHA512_ASM -DVPAES_ASM -DX25519_ASM -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/common/der/libfips-lib-der_rsa_sig.d.tmp -MT providers/common/der/libfips-lib-der_rsa_sig.o -c -o providers/common/der/libfips-lib-der_rsa_sig.o providers/common/der/der_rsa_sig.c
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -DAES_ASM -DBSAES_ASM -DECP_NISTZ256_ASM -DFIPS_MODULE -DGHASH_ASM -DKECCAK1600_ASM -DOPENSSL_BN_ASM_GF2m -DOPENSSL_BN_ASM_MONT -DOPENSSL_BN_ASM_MONT5 -DOPENSSL_CPUID_OBJ -DOPENSSL_IA32_SSE2 -DSHA1_ASM -DSHA256_ASM -DSHA512_ASM -DVPAES_ASM -DX25519_ASM -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/common/der/libfips-lib-der_sm2_gen.d.tmp -MT providers/common/der/libfips-lib-der_sm2_gen.o -c -o providers/common/der/libfips-lib-der_sm2_gen.o providers/common/der/der_sm2_gen.c
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -DAES_ASM -DBSAES_ASM -DECP_NISTZ256_ASM -DFIPS_MODULE -DGHASH_ASM -DKECCAK1600_ASM -DOPENSSL_BN_ASM_GF2m -DOPENSSL_BN_ASM_MONT -DOPENSSL_BN_ASM_MONT5 -DOPENSSL_CPUID_OBJ -DOPENSSL_IA32_SSE2 -DSHA1_ASM -DSHA256_ASM -DSHA512_ASM -DVPAES_ASM -DX25519_ASM -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/common/der/libfips-lib-der_sm2_key.d.tmp -MT providers/common/der/libfips-lib-der_sm2_key.o -c -o providers/common/der/libfips-lib-der_sm2_key.o providers/common/der/der_sm2_key.c
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -DAES_ASM -DBSAES_ASM -DECP_NISTZ256_ASM -DFIPS_MODULE -DGHASH_ASM -DKECCAK1600_ASM -DOPENSSL_BN_ASM_GF2m -DOPENSSL_BN_ASM_MONT -DOPENSSL_BN_ASM_MONT5 -DOPENSSL_CPUID_OBJ -DOPENSSL_IA32_SSE2 -DSHA1_ASM -DSHA256_ASM -DSHA512_ASM -DVPAES_ASM -DX25519_ASM -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/common/der/libfips-lib-der_sm2_sig.d.tmp -MT providers/common/der/libfips-lib-der_sm2_sig.o -c -o providers/common/der/libfips-lib-der_sm2_sig.o providers/common/der/der_sm2_sig.c
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -DAES_ASM -DBSAES_ASM -DECP_NISTZ256_ASM -DFIPS_MODULE -DGHASH_ASM -DKECCAK1600_ASM -DOPENSSL_BN_ASM_GF2m -DOPENSSL_BN_ASM_MONT -DOPENSSL_BN_ASM_MONT5 -DOPENSSL_CPUID_OBJ -DOPENSSL_IA32_SSE2 -DSHA1_ASM -DSHA256_ASM -DSHA512_ASM -DVPAES_ASM -DX25519_ASM -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/common/der/libfips-lib-der_wrap_gen.d.tmp -MT providers/common/der/libfips-lib-der_wrap_gen.o -c -o providers/common/der/libfips-lib-der_wrap_gen.o providers/common/der/der_wrap_gen.c
ccache clang  -Iproviders/common/include/prov -I. -Icrypto -Iinclude -Iproviders/implementations/include -Iproviders/common/include  -DAES_ASM -DBSAES_ASM -DECP_NISTZ256_ASM -DFIPS_MODULE -DGHASH_ASM -DKECCAK1600_ASM -DOPENSSL_BN_ASM_GF2m -DOPENSSL_BN_ASM_MONT -DOPENSSL_BN_ASM_MONT5 -DOPENSSL_CPUID_OBJ -DOPENSSL_IA32_SSE2 -DSHA1_ASM -DSHA256_ASM -DSHA512_ASM -DVPAES_ASM -DX25519_ASM -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF providers/implementations/signature/libfips-lib-rsa.d.tmp -MT providers/implementations/signature/libfips-lib-rsa.o -c -o providers/implementations/signature/libfips-lib-rsa.o providers/implementations/signature/rsa.c
rm -f providers/libfips.a
ar qc providers/libfips.a crypto/aes/libfips-lib-aes-x86_64.o crypto/aes/libfips-lib-aes_ecb.o crypto/aes/libfips-lib-aes_misc.o crypto/aes/libfips-lib-aesni-mb-x86_64.o crypto/aes/libfips-lib-aesni-sha1-x86_64.o crypto/aes/libfips-lib-aesni-sha256-x86_64.o crypto/aes/libfips-lib-aesni-x86_64.o crypto/aes/libfips-lib-bsaes-x86_64.o crypto/aes/libfips-lib-vpaes-x86_64.o crypto/bn/asm/libfips-lib-x86_64-gcc.o crypto/bn/libfips-lib-bn_add.o crypto/bn/libfips-lib-bn_blind.o crypto/bn/libfips-lib-bn_const.o crypto/bn/libfips-lib-bn_conv.o crypto/bn/libfips-lib-bn_ctx.o crypto/bn/libfips-lib-bn_dh.o crypto/bn/libfips-lib-bn_div.o crypto/bn/libfips-lib-bn_exp.o crypto/bn/libfips-lib-bn_exp2.o crypto/bn/libfips-lib-bn_gcd.o crypto/bn/libfips-lib-bn_gf2m.o crypto/bn/libfips-lib-bn_intern.o crypto/bn/libfips-lib-bn_kron.o crypto/bn/libfips-lib-bn_lib.o crypto/bn/libfips-lib-bn_mod.o crypto/bn/libfips-lib-bn_mont.o crypto/bn/libfips-lib-bn_mpi.o crypto/bn/libfips-lib-bn_mul.o crypto/bn/libfips-lib-bn_nist.o crypto/bn/libfips-lib-bn_prime.o crypto/bn/libfips-lib-bn_rand.o crypto/bn/libfips-lib-bn_recp.o crypto/bn/libfips-lib-bn_rsa_fips186_4.o crypto/bn/libfips-lib-bn_shift.o crypto/bn/libfips-lib-bn_sqr.o crypto/bn/libfips-lib-bn_sqrt.o crypto/bn/libfips-lib-bn_word.o crypto/bn/libfips-lib-rsaz-avx2.o crypto/bn/libfips-lib-rsaz-avx512.o crypto/bn/libfips-lib-rsaz-x86_64.o crypto/bn/libfips-lib-rsaz_exp.o crypto/bn/libfips-lib-rsaz_exp_x2.o crypto/bn/libfips-lib-x86_64-gf2m.o crypto/bn/libfips-lib-x86_64-mont.o crypto/bn/libfips-lib-x86_64-mont5.o crypto/buffer/libfips-lib-buffer.o crypto/cmac/libfips-lib-cmac.o crypto/des/libfips-lib-des_enc.o crypto/des/libfips-lib-ecb3_enc.o crypto/des/libfips-lib-fcrypt_b.o crypto/des/libfips-lib-set_key.o crypto/dh/libfips-lib-dh_backend.o crypto/dh/libfips-lib-dh_check.o crypto/dh/libfips-lib-dh_gen.o crypto/dh/libfips-lib-dh_group_params.o crypto/dh/libfips-lib-dh_kdf.o crypto/dh/libfips-lib-dh_key.o crypto/dh/libfips-lib-dh_lib.o crypto/dsa/libfips-lib-dsa_backend.o crypto/dsa/libfips-lib-dsa_check.o crypto/dsa/libfips-lib-dsa_gen.o crypto/dsa/libfips-lib-dsa_key.o crypto/dsa/libfips-lib-dsa_lib.o crypto/dsa/libfips-lib-dsa_ossl.o crypto/dsa/libfips-lib-dsa_sign.o crypto/dsa/libfips-lib-dsa_vrf.o crypto/ec/curve448/arch_32/libfips-lib-f_impl32.o crypto/ec/curve448/arch_64/libfips-lib-f_impl64.o crypto/ec/curve448/libfips-lib-curve448.o crypto/ec/curve448/libfips-lib-curve448_tables.o crypto/ec/curve448/libfips-lib-eddsa.o crypto/ec/curve448/libfips-lib-f_generic.o crypto/ec/curve448/libfips-lib-scalar.o crypto/ec/libfips-lib-curve25519.o crypto/ec/libfips-lib-ec2_oct.o crypto/ec/libfips-lib-ec2_smpl.o crypto/ec/libfips-lib-ec_asn1.o crypto/ec/libfips-lib-ec_backend.o crypto/ec/libfips-lib-ec_check.o crypto/ec/libfips-lib-ec_curve.o crypto/ec/libfips-lib-ec_cvt.o crypto/ec/libfips-lib-ec_deprecated.o crypto/ec/libfips-lib-ec_key.o crypto/ec/libfips-lib-ec_kmeth.o crypto/ec/libfips-lib-ec_lib.o crypto/ec/libfips-lib-ec_mult.o crypto/ec/libfips-lib-ec_oct.o crypto/ec/libfips-lib-ec_print.o crypto/ec/libfips-lib-ecdh_kdf.o crypto/ec/libfips-lib-ecdh_ossl.o crypto/ec/libfips-lib-ecdsa_ossl.o crypto/ec/libfips-lib-ecdsa_sign.o crypto/ec/libfips-lib-ecdsa_vrf.o crypto/ec/libfips-lib-ecp_mont.o crypto/ec/libfips-lib-ecp_nist.o crypto/ec/libfips-lib-ecp_nistp224.o crypto/ec/libfips-lib-ecp_nistp256.o crypto/ec/libfips-lib-ecp_nistp521.o crypto/ec/libfips-lib-ecp_nistputil.o crypto/ec/libfips-lib-ecp_nistz256-x86_64.o crypto/ec/libfips-lib-ecp_nistz256.o crypto/ec/libfips-lib-ecp_oct.o crypto/ec/libfips-lib-ecp_smpl.o crypto/ec/libfips-lib-ecx_backend.o crypto/ec/libfips-lib-ecx_key.o crypto/ec/libfips-lib-x25519-x86_64.o crypto/evp/libfips-lib-asymcipher.o crypto/evp/libfips-lib-cmeth_lib.o crypto/evp/libfips-lib-dh_support.o crypto/evp/libfips-lib-digest.o crypto/evp/libfips-lib-ec_support.o crypto/evp/libfips-lib-evp_enc.o crypto/evp/libfips-lib-evp_fetch.o crypto/evp/libfips-lib-evp_lib.o crypto/evp/libfips-lib-evp_rand.o crypto/evp/libfips-lib-evp_utils.o crypto/evp/libfips-lib-exchange.o crypto/evp/libfips-lib-kdf_lib.o crypto/evp/libfips-lib-kdf_meth.o crypto/evp/libfips-lib-kem.o crypto/evp/libfips-lib-keymgmt_lib.o crypto/evp/libfips-lib-keymgmt_meth.o crypto/evp/libfips-lib-m_sigver.o crypto/evp/libfips-lib-mac_lib.o crypto/evp/libfips-lib-mac_meth.o crypto/evp/libfips-lib-p_lib.o crypto/evp/libfips-lib-pmeth_check.o crypto/evp/libfips-lib-pmeth_gn.o crypto/evp/libfips-lib-pmeth_lib.o crypto/evp/libfips-lib-signature.o crypto/ffc/libfips-lib-ffc_backend.o crypto/ffc/libfips-lib-ffc_dh.o crypto/ffc/libfips-lib-ffc_key_generate.o crypto/ffc/libfips-lib-ffc_key_validate.o crypto/ffc/libfips-lib-ffc_params.o crypto/ffc/libfips-lib-ffc_params_generate.o crypto/ffc/libfips-lib-ffc_params_validate.o crypto/hmac/libfips-lib-hmac.o crypto/lhash/libfips-lib-lhash.o crypto/libfips-lib-asn1_dsa.o crypto/libfips-lib-bsearch.o crypto/libfips-lib-context.o crypto/libfips-lib-core_algorithm.o crypto/libfips-lib-core_fetch.o crypto/libfips-lib-core_namemap.o crypto/libfips-lib-cpuid.o crypto/libfips-lib-cryptlib.o crypto/libfips-lib-ctype.o crypto/libfips-lib-der_writer.o crypto/libfips-lib-ex_data.o crypto/libfips-lib-initthread.o crypto/libfips-lib-o_str.o crypto/libfips-lib-packet.o crypto/libfips-lib-param_build.o crypto/libfips-lib-param_build_set.o crypto/libfips-lib-params.o crypto/libfips-lib-params_dup.o crypto/libfips-lib-params_from_text.o crypto/libfips-lib-passphrase.o crypto/libfips-lib-provider_core.o crypto/libfips-lib-provider_predefined.o crypto/libfips-lib-self_test_core.o crypto/libfips-lib-sparse_array.o crypto/libfips-lib-threads_lib.o crypto/libfips-lib-threads_none.o crypto/libfips-lib-threads_pthread.o crypto/libfips-lib-threads_win.o crypto/libfips-lib-x86_64cpuid.o crypto/modes/libfips-lib-aesni-gcm-x86_64.o crypto/modes/libfips-lib-cbc128.o crypto/modes/libfips-lib-ccm128.o crypto/modes/libfips-lib-cfb128.o crypto/modes/libfips-lib-ctr128.o crypto/modes/libfips-lib-gcm128.o crypto/modes/libfips-lib-ghash-x86_64.o crypto/modes/libfips-lib-ofb128.o crypto/modes/libfips-lib-wrap128.o crypto/modes/libfips-lib-xts128.o crypto/property/libfips-lib-defn_cache.o crypto/property/libfips-lib-property.o crypto/property/libfips-lib-property_parse.o crypto/property/libfips-lib-property_string.o crypto/rand/libfips-lib-rand_lib.o crypto/rand/libfips-lib-rand_meth.o crypto/rsa/libfips-lib-rsa_acvp_test_params.o crypto/rsa/libfips-lib-rsa_backend.o crypto/rsa/libfips-lib-rsa_chk.o crypto/rsa/libfips-lib-rsa_crpt.o crypto/rsa/libfips-lib-rsa_gen.o crypto/rsa/libfips-lib-rsa_lib.o crypto/rsa/libfips-lib-rsa_mp_names.o crypto/rsa/libfips-lib-rsa_none.o crypto/rsa/libfips-lib-rsa_oaep.o crypto/rsa/libfips-lib-rsa_ossl.o crypto/rsa/libfips-lib-rsa_pk1.o crypto/rsa/libfips-lib-rsa_pss.o crypto/rsa/libfips-lib-rsa_schemes.o crypto/rsa/libfips-lib-rsa_sign.o crypto/rsa/libfips-lib-rsa_sp800_56b_check.o crypto/rsa/libfips-lib-rsa_sp800_56b_gen.o crypto/rsa/libfips-lib-rsa_x931.o crypto/sha/libfips-lib-keccak1600-x86_64.o crypto/sha/libfips-lib-sha1-mb-x86_64.o crypto/sha/libfips-lib-sha1-x86_64.o crypto/sha/libfips-lib-sha1dgst.o crypto/sha/libfips-lib-sha256-mb-x86_64.o crypto/sha/libfips-lib-sha256-x86_64.o crypto/sha/libfips-lib-sha256.o crypto/sha/libfips-lib-sha3.o crypto/sha/libfips-lib-sha512-x86_64.o crypto/sha/libfips-lib-sha512.o crypto/stack/libfips-lib-stack.o providers/common/der/libfips-lib-der_digests_gen.o providers/common/der/libfips-lib-der_dsa_gen.o providers/common/der/libfips-lib-der_dsa_key.o providers/common/der/libfips-lib-der_dsa_sig.o providers/common/der/libfips-lib-der_ec_gen.o providers/common/der/libfips-lib-der_ec_key.o providers/common/der/libfips-lib-der_ec_sig.o providers/common/der/libfips-lib-der_ecx_gen.o providers/common/der/libfips-lib-der_ecx_key.o providers/common/der/libfips-lib-der_rsa_gen.o providers/common/der/libfips-lib-der_rsa_key.o providers/common/der/libfips-lib-der_rsa_sig.o providers/common/der/libfips-lib-der_sm2_gen.o providers/common/der/libfips-lib-der_sm2_key.o providers/common/der/libfips-lib-der_sm2_sig.o providers/common/der/libfips-lib-der_wrap_gen.o providers/common/libfips-lib-bio_prov.o providers/common/libfips-lib-capabilities.o providers/common/libfips-lib-digest_to_nid.o providers/common/libfips-lib-provider_seeding.o providers/common/libfips-lib-provider_util.o providers/common/libfips-lib-securitycheck.o providers/common/libfips-lib-securitycheck_fips.o providers/implementations/ciphers/libfips-lib-cipher_aes_xts_fips.o providers/implementations/kdfs/libfips-lib-pbkdf2_fips.o providers/implementations/keymgmt/libfips-lib-dh_kmgmt.o providers/implementations/keymgmt/libfips-lib-dsa_kmgmt.o providers/implementations/keymgmt/libfips-lib-ec_kmgmt.o providers/implementations/keymgmt/libfips-lib-mac_legacy_kmgmt.o providers/implementations/keymgmt/libfips-lib-rsa_kmgmt.o providers/implementations/rands/libfips-lib-crngt.o providers/implementations/rands/libfips-lib-drbg.o providers/implementations/rands/libfips-lib-drbg_ctr.o providers/implementations/rands/libfips-lib-drbg_hash.o providers/implementations/rands/libfips-lib-drbg_hmac.o providers/implementations/rands/libfips-lib-test_rng.o providers/implementations/signature/libfips-lib-mac_legacy.o providers/implementations/signature/libfips-lib-rsa.o
ranlib providers/libfips.a || echo Never mind.
rm -f providers/libimplementations.a
ar qc providers/libimplementations.a crypto/md5/libimplementations-lib-md5-x86_64.o crypto/md5/libimplementations-lib-md5_dgst.o crypto/md5/libimplementations-lib-md5_one.o crypto/md5/libimplementations-lib-md5_sha1.o providers/implementations/asymciphers/libimplementations-lib-rsa_enc.o providers/implementations/asymciphers/libimplementations-lib-sm2_enc.o providers/implementations/ciphers/libimplementations-lib-cipher_aes.o providers/implementations/ciphers/libimplementations-lib-cipher_aes_cbc_hmac_sha.o providers/implementations/ciphers/libimplementations-lib-cipher_aes_cbc_hmac_sha1_hw.o providers/implementations/ciphers/libimplementations-lib-cipher_aes_cbc_hmac_sha256_hw.o providers/implementations/ciphers/libimplementations-lib-cipher_aes_ccm.o providers/implementations/ciphers/libimplementations-lib-cipher_aes_ccm_hw.o providers/implementations/ciphers/libimplementations-lib-cipher_aes_cts.o providers/implementations/ciphers/libimplementations-lib-cipher_aes_gcm.o providers/implementations/ciphers/libimplementations-lib-cipher_aes_gcm_hw.o providers/implementations/ciphers/libimplementations-lib-cipher_aes_hw.o providers/implementations/ciphers/libimplementations-lib-cipher_aes_ocb.o providers/implementations/ciphers/libimplementations-lib-cipher_aes_ocb_hw.o providers/implementations/ciphers/libimplementations-lib-cipher_aes_siv.o providers/implementations/ciphers/libimplementations-lib-cipher_aes_siv_hw.o providers/implementations/ciphers/libimplementations-lib-cipher_aes_wrp.o providers/implementations/ciphers/libimplementations-lib-cipher_aes_xts.o providers/implementations/ciphers/libimplementations-lib-cipher_aes_xts_hw.o providers/implementations/ciphers/libimplementations-lib-cipher_aria.o providers/implementations/ciphers/libimplementations-lib-cipher_aria_ccm.o providers/implementations/ciphers/libimplementations-lib-cipher_aria_ccm_hw.o providers/implementations/ciphers/libimplementations-lib-cipher_aria_gcm.o providers/implementations/ciphers/libimplementations-lib-cipher_aria_gcm_hw.o providers/implementations/ciphers/libimplementations-lib-cipher_aria_hw.o providers/implementations/ciphers/libimplementations-lib-cipher_camellia.o providers/implementations/ciphers/libimplementations-lib-cipher_camellia_hw.o providers/implementations/ciphers/libimplementations-lib-cipher_chacha20.o providers/implementations/ciphers/libimplementations-lib-cipher_chacha20_hw.o providers/implementations/ciphers/libimplementations-lib-cipher_chacha20_poly1305.o providers/implementations/ciphers/libimplementations-lib-cipher_chacha20_poly1305_hw.o providers/implementations/ciphers/libimplementations-lib-cipher_null.o providers/implementations/ciphers/libimplementations-lib-cipher_sm4.o providers/implementations/ciphers/libimplementations-lib-cipher_sm4_hw.o providers/implementations/ciphers/libimplementations-lib-cipher_tdes.o providers/implementations/ciphers/libimplementations-lib-cipher_tdes_common.o providers/implementations/ciphers/libimplementations-lib-cipher_tdes_default.o providers/implementations/ciphers/libimplementations-lib-cipher_tdes_default_hw.o providers/implementations/ciphers/libimplementations-lib-cipher_tdes_hw.o providers/implementations/ciphers/libimplementations-lib-cipher_tdes_wrap.o providers/implementations/ciphers/libimplementations-lib-cipher_tdes_wrap_hw.o providers/implementations/digests/libimplementations-lib-blake2_prov.o providers/implementations/digests/libimplementations-lib-blake2b_prov.o providers/implementations/digests/libimplementations-lib-blake2s_prov.o providers/implementations/digests/libimplementations-lib-md5_prov.o providers/implementations/digests/libimplementations-lib-md5_sha1_prov.o providers/implementations/digests/libimplementations-lib-sha2_prov.o providers/implementations/digests/libimplementations-lib-sha3_prov.o providers/implementations/digests/libimplementations-lib-sm3_prov.o providers/implementations/encode_decode/libimplementations-lib-decode_der2key.o providers/implementations/encode_decode/libimplementations-lib-decode_msblob2key.o providers/implementations/encode_decode/libimplementations-lib-decode_pem2der.o providers/implementations/encode_decode/libimplementations-lib-decode_pvk2key.o providers/implementations/encode_decode/libimplementations-lib-encode_key2any.o providers/implementations/encode_decode/libimplementations-lib-encode_key2blob.o providers/implementations/encode_decode/libimplementations-lib-encode_key2ms.o providers/implementations/encode_decode/libimplementations-lib-encode_key2text.o providers/implementations/encode_decode/libimplementations-lib-endecoder_common.o providers/implementations/exchange/libimplementations-lib-dh_exch.o providers/implementations/exchange/libimplementations-lib-ecdh_exch.o providers/implementations/exchange/libimplementations-lib-ecx_exch.o providers/implementations/exchange/libimplementations-lib-kdf_exch.o providers/implementations/kdfs/libimplementations-lib-hkdf.o providers/implementations/kdfs/libimplementations-lib-kbkdf.o providers/implementations/kdfs/libimplementations-lib-krb5kdf.o providers/implementations/kdfs/libimplementations-lib-pbkdf2.o providers/implementations/kdfs/libimplementations-lib-pkcs12kdf.o providers/implementations/kdfs/libimplementations-lib-scrypt.o providers/implementations/kdfs/libimplementations-lib-sshkdf.o providers/implementations/kdfs/libimplementations-lib-sskdf.o providers/implementations/kdfs/libimplementations-lib-tls1_prf.o providers/implementations/kdfs/libimplementations-lib-x942kdf.o providers/implementations/kem/libimplementations-lib-rsa_kem.o providers/implementations/keymgmt/libimplementations-lib-ecx_kmgmt.o providers/implementations/keymgmt/libimplementations-lib-kdf_legacy_kmgmt.o providers/implementations/macs/libimplementations-lib-blake2b_mac.o providers/implementations/macs/libimplementations-lib-blake2s_mac.o providers/implementations/macs/libimplementations-lib-cmac_prov.o providers/implementations/macs/libimplementations-lib-gmac_prov.o providers/implementations/macs/libimplementations-lib-hmac_prov.o providers/implementations/macs/libimplementations-lib-kmac_prov.o providers/implementations/macs/libimplementations-lib-poly1305_prov.o providers/implementations/macs/libimplementations-lib-siphash_prov.o providers/implementations/signature/libimplementations-lib-dsa.o providers/implementations/signature/libimplementations-lib-ecdsa.o providers/implementations/signature/libimplementations-lib-eddsa.o providers/implementations/signature/libimplementations-lib-sm2sig.o providers/implementations/storemgmt/libimplementations-lib-file_store.o providers/implementations/storemgmt/libimplementations-lib-file_store_der2obj.o ssl/libimplementations-lib-s3_cbc.o
ranlib providers/libimplementations.a || echo Never mind.
rm -f providers/libnonfips.a
ar qc providers/libnonfips.a providers/common/der/libnonfips-lib-der_digests_gen.o providers/common/der/libnonfips-lib-der_dsa_gen.o providers/common/der/libnonfips-lib-der_dsa_key.o providers/common/der/libnonfips-lib-der_dsa_sig.o providers/common/der/libnonfips-lib-der_ec_gen.o providers/common/der/libnonfips-lib-der_ec_key.o providers/common/der/libnonfips-lib-der_ec_sig.o providers/common/der/libnonfips-lib-der_ecx_gen.o providers/common/der/libnonfips-lib-der_ecx_key.o providers/common/der/libnonfips-lib-der_rsa_gen.o providers/common/der/libnonfips-lib-der_rsa_key.o providers/common/der/libnonfips-lib-der_rsa_sig.o providers/common/der/libnonfips-lib-der_sm2_gen.o providers/common/der/libnonfips-lib-der_sm2_key.o providers/common/der/libnonfips-lib-der_sm2_sig.o providers/common/der/libnonfips-lib-der_wrap_gen.o providers/common/libnonfips-lib-bio_prov.o providers/common/libnonfips-lib-capabilities.o providers/common/libnonfips-lib-digest_to_nid.o providers/common/libnonfips-lib-provider_seeding.o providers/common/libnonfips-lib-provider_util.o providers/common/libnonfips-lib-securitycheck.o providers/common/libnonfips-lib-securitycheck_default.o providers/implementations/ciphers/libnonfips-lib-cipher_aes_xts_fips.o providers/implementations/kdfs/libnonfips-lib-pbkdf2_fips.o providers/implementations/keymgmt/libnonfips-lib-dh_kmgmt.o providers/implementations/keymgmt/libnonfips-lib-dsa_kmgmt.o providers/implementations/keymgmt/libnonfips-lib-ec_kmgmt.o providers/implementations/keymgmt/libnonfips-lib-mac_legacy_kmgmt.o providers/implementations/keymgmt/libnonfips-lib-rsa_kmgmt.o providers/implementations/rands/libnonfips-lib-crngt.o providers/implementations/rands/libnonfips-lib-drbg.o providers/implementations/rands/libnonfips-lib-drbg_ctr.o providers/implementations/rands/libnonfips-lib-drbg_hash.o providers/implementations/rands/libnonfips-lib-drbg_hmac.o providers/implementations/rands/libnonfips-lib-seed_src.o providers/implementations/rands/libnonfips-lib-test_rng.o providers/implementations/rands/seeding/libnonfips-lib-rand_cpu_x86.o providers/implementations/rands/seeding/libnonfips-lib-rand_tsc.o providers/implementations/rands/seeding/libnonfips-lib-rand_unix.o providers/implementations/rands/seeding/libnonfips-lib-rand_win.o providers/implementations/signature/libnonfips-lib-mac_legacy.o providers/implementations/signature/libnonfips-lib-rsa.o providers/libnonfips-lib-prov_running.o
ranlib providers/libnonfips.a || echo Never mind.
ccache clang  -Iinclude -Iapps/include -I.  -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF apps/lib/libtestutil-lib-opt.d.tmp -MT apps/lib/libtestutil-lib-opt.o -c -o apps/lib/libtestutil-lib-opt.o apps/lib/opt.c
rm -f test/libtestutil.a
ar qc test/libtestutil.a apps/lib/libtestutil-lib-opt.o test/testutil/libtestutil-lib-apps_mem.o test/testutil/libtestutil-lib-basic_output.o test/testutil/libtestutil-lib-cb.o test/testutil/libtestutil-lib-driver.o test/testutil/libtestutil-lib-fake_random.o test/testutil/libtestutil-lib-format_output.o test/testutil/libtestutil-lib-load.o test/testutil/libtestutil-lib-main.o test/testutil/libtestutil-lib-options.o test/testutil/libtestutil-lib-output.o test/testutil/libtestutil-lib-provider.o test/testutil/libtestutil-lib-random.o test/testutil/libtestutil-lib-stanza.o test/testutil/libtestutil-lib-test_cleanup.o test/testutil/libtestutil-lib-test_options.o test/testutil/libtestutil-lib-tests.o test/testutil/libtestutil-lib-testutil_init.o
ranlib test/libtestutil.a || echo Never mind.
ccache clang  -I. -Iinclude -Iproviders/common/include -Iproviders/implementations/include  -DAES_ASM -DBSAES_ASM -DCMLL_ASM -DECP_NISTZ256_ASM -DGHASH_ASM -DKECCAK1600_ASM -DMD5_ASM -DOPENSSL_BN_ASM_GF2m -DOPENSSL_BN_ASM_MONT -DOPENSSL_BN_ASM_MONT5 -DOPENSSL_CPUID_OBJ -DOPENSSL_IA32_SSE2 -DPOLY1305_ASM -DSHA1_ASM -DSHA256_ASM -DSHA512_ASM -DVPAES_ASM -DWHIRLPOOL_ASM -DX25519_ASM -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF crypto/evp/libcrypto-shlib-keymgmt_lib.d.tmp -MT crypto/evp/libcrypto-shlib-keymgmt_lib.o -c -o crypto/evp/libcrypto-shlib-keymgmt_lib.o crypto/evp/keymgmt_lib.c
ccache clang  -I. -Iinclude -Iproviders/common/include -Iproviders/implementations/include  -DAES_ASM -DBSAES_ASM -DCMLL_ASM -DECP_NISTZ256_ASM -DGHASH_ASM -DKECCAK1600_ASM -DMD5_ASM -DOPENSSL_BN_ASM_GF2m -DOPENSSL_BN_ASM_MONT -DOPENSSL_BN_ASM_MONT5 -DOPENSSL_CPUID_OBJ -DOPENSSL_IA32_SSE2 -DPOLY1305_ASM -DSHA1_ASM -DSHA256_ASM -DSHA512_ASM -DVPAES_ASM -DWHIRLPOOL_ASM -DX25519_ASM -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF crypto/evp/libcrypto-shlib-p_lib.d.tmp -MT crypto/evp/libcrypto-shlib-p_lib.o -c -o crypto/evp/libcrypto-shlib-p_lib.o crypto/evp/p_lib.c
ccache clang  -I. -Iinclude -Iproviders/common/include -Iproviders/implementations/include  -DAES_ASM -DBSAES_ASM -DCMLL_ASM -DECP_NISTZ256_ASM -DGHASH_ASM -DKECCAK1600_ASM -DMD5_ASM -DOPENSSL_BN_ASM_GF2m -DOPENSSL_BN_ASM_MONT -DOPENSSL_BN_ASM_MONT5 -DOPENSSL_CPUID_OBJ -DOPENSSL_IA32_SSE2 -DPOLY1305_ASM -DSHA1_ASM -DSHA256_ASM -DSHA512_ASM -DVPAES_ASM -DWHIRLPOOL_ASM -DX25519_ASM -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -DOPENSSL_BUILDING_OPENSSL  -MMD -MF crypto/store/libcrypto-shlib-store_result.d.tmp -MT crypto/store/libcrypto-shlib-store_result.o -c -o crypto/store/libcrypto-shlib-store_result.o crypto/store/store_result.c
ccache clang -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -Wl,-znodelete -shared -Wl,-Bsymbolic   -Wl,-soname=libcrypto.so.3 \
	-o libcrypto.so.3 -Wl,--version-script=libcrypto.ld \
	crypto/aes/libcrypto-shlib-aes-x86_64.o \
	crypto/aes/libcrypto-shlib-aes_cfb.o \
	crypto/aes/libcrypto-shlib-aes_ecb.o \
	crypto/aes/libcrypto-shlib-aes_ige.o \
	crypto/aes/libcrypto-shlib-aes_misc.o \
	crypto/aes/libcrypto-shlib-aes_ofb.o \
	crypto/aes/libcrypto-shlib-aes_wrap.o \
	crypto/aes/libcrypto-shlib-aesni-mb-x86_64.o \
	crypto/aes/libcrypto-shlib-aesni-sha1-x86_64.o \
	crypto/aes/libcrypto-shlib-aesni-sha256-x86_64.o \
	crypto/aes/libcrypto-shlib-aesni-x86_64.o \
	crypto/aes/libcrypto-shlib-bsaes-x86_64.o \
	crypto/aes/libcrypto-shlib-vpaes-x86_64.o \
	crypto/aria/libcrypto-shlib-aria.o \
	crypto/asn1/libcrypto-shlib-a_bitstr.o \
	crypto/asn1/libcrypto-shlib-a_d2i_fp.o \
	crypto/asn1/libcrypto-shlib-a_digest.o \
	crypto/asn1/libcrypto-shlib-a_dup.o \
	crypto/asn1/libcrypto-shlib-a_gentm.o \
	crypto/asn1/libcrypto-shlib-a_i2d_fp.o \
	crypto/asn1/libcrypto-shlib-a_int.o \
	crypto/asn1/libcrypto-shlib-a_mbstr.o \
	crypto/asn1/libcrypto-shlib-a_object.o \
	crypto/asn1/libcrypto-shlib-a_octet.o \
	crypto/asn1/libcrypto-shlib-a_print.o \
	crypto/asn1/libcrypto-shlib-a_sign.o \
	crypto/asn1/libcrypto-shlib-a_strex.o \
	crypto/asn1/libcrypto-shlib-a_strnid.o \
	crypto/asn1/libcrypto-shlib-a_time.o \
	crypto/asn1/libcrypto-shlib-a_type.o \
	crypto/asn1/libcrypto-shlib-a_utctm.o \
	crypto/asn1/libcrypto-shlib-a_utf8.o \
	crypto/asn1/libcrypto-shlib-a_verify.o \
	crypto/asn1/libcrypto-shlib-ameth_lib.o \
	crypto/asn1/libcrypto-shlib-asn1_err.o \
	crypto/asn1/libcrypto-shlib-asn1_gen.o \
	crypto/asn1/libcrypto-shlib-asn1_item_list.o \
	crypto/asn1/libcrypto-shlib-asn1_lib.o \
	crypto/asn1/libcrypto-shlib-asn1_par.o \
	crypto/asn1/libcrypto-shlib-asn_mime.o \
	crypto/asn1/libcrypto-shlib-asn_moid.o \
	crypto/asn1/libcrypto-shlib-asn_mstbl.o \
	crypto/asn1/libcrypto-shlib-asn_pack.o \
	crypto/asn1/libcrypto-shlib-bio_asn1.o \
	crypto/asn1/libcrypto-shlib-bio_ndef.o \
	crypto/asn1/libcrypto-shlib-d2i_param.o \
	crypto/asn1/libcrypto-shlib-d2i_pr.o \
	crypto/asn1/libcrypto-shlib-d2i_pu.o \
	crypto/asn1/libcrypto-shlib-evp_asn1.o \
	crypto/asn1/libcrypto-shlib-f_int.o \
	crypto/asn1/libcrypto-shlib-f_string.o \
	crypto/asn1/libcrypto-shlib-i2d_evp.o \
	crypto/asn1/libcrypto-shlib-n_pkey.o \
	crypto/asn1/libcrypto-shlib-nsseq.o \
	crypto/asn1/libcrypto-shlib-p5_pbe.o \
	crypto/asn1/libcrypto-shlib-p5_pbev2.o \
	crypto/asn1/libcrypto-shlib-p5_scrypt.o \
	crypto/asn1/libcrypto-shlib-p8_pkey.o \
	crypto/asn1/libcrypto-shlib-t_bitst.o \
	crypto/asn1/libcrypto-shlib-t_pkey.o \
	crypto/asn1/libcrypto-shlib-t_spki.o \
	crypto/asn1/libcrypto-shlib-tasn_dec.o \
	crypto/asn1/libcrypto-shlib-tasn_enc.o \
	crypto/asn1/libcrypto-shlib-tasn_fre.o \
	crypto/asn1/libcrypto-shlib-tasn_new.o \
	crypto/asn1/libcrypto-shlib-tasn_prn.o \
	crypto/asn1/libcrypto-shlib-tasn_scn.o \
	crypto/asn1/libcrypto-shlib-tasn_typ.o \
	crypto/asn1/libcrypto-shlib-tasn_utl.o \
	crypto/asn1/libcrypto-shlib-x_algor.o \
	crypto/asn1/libcrypto-shlib-x_bignum.o \
	crypto/asn1/libcrypto-shlib-x_info.o \
	crypto/asn1/libcrypto-shlib-x_int64.o \
	crypto/asn1/libcrypto-shlib-x_long.o \
	crypto/asn1/libcrypto-shlib-x_pkey.o \
	crypto/asn1/libcrypto-shlib-x_sig.o \
	crypto/asn1/libcrypto-shlib-x_spki.o \
	crypto/asn1/libcrypto-shlib-x_val.o \
	crypto/async/arch/libcrypto-shlib-async_null.o \
	crypto/async/arch/libcrypto-shlib-async_posix.o \
	crypto/async/arch/libcrypto-shlib-async_win.o \
	crypto/async/libcrypto-shlib-async.o \
	crypto/async/libcrypto-shlib-async_err.o \
	crypto/async/libcrypto-shlib-async_wait.o \
	crypto/bf/libcrypto-shlib-bf_cfb64.o \
	crypto/bf/libcrypto-shlib-bf_ecb.o \
	crypto/bf/libcrypto-shlib-bf_enc.o \
	crypto/bf/libcrypto-shlib-bf_ofb64.o \
	crypto/bf/libcrypto-shlib-bf_skey.o \
	crypto/bio/libcrypto-shlib-b_addr.o \
	crypto/bio/libcrypto-shlib-b_dump.o \
	crypto/bio/libcrypto-shlib-b_print.o \
	crypto/bio/libcrypto-shlib-b_sock.o \
	crypto/bio/libcrypto-shlib-b_sock2.o \
	crypto/bio/libcrypto-shlib-bf_buff.o \
	crypto/bio/libcrypto-shlib-bf_lbuf.o \
	crypto/bio/libcrypto-shlib-bf_nbio.o \
	crypto/bio/libcrypto-shlib-bf_null.o \
	crypto/bio/libcrypto-shlib-bf_prefix.o \
	crypto/bio/libcrypto-shlib-bf_readbuff.o \
	crypto/bio/libcrypto-shlib-bio_cb.o \
	crypto/bio/libcrypto-shlib-bio_err.o \
	crypto/bio/libcrypto-shlib-bio_lib.o \
	crypto/bio/libcrypto-shlib-bio_meth.o \
	crypto/bio/libcrypto-shlib-bss_acpt.o \
	crypto/bio/libcrypto-shlib-bss_bio.o \
	crypto/bio/libcrypto-shlib-bss_conn.o \
	crypto/bio/libcrypto-shlib-bss_dgram.o \
	crypto/bio/libcrypto-shlib-bss_fd.o \
	crypto/bio/libcrypto-shlib-bss_file.o \
	crypto/bio/libcrypto-shlib-bss_log.o \
	crypto/bio/libcrypto-shlib-bss_mem.o \
	crypto/bio/libcrypto-shlib-bss_null.o \
	crypto/bio/libcrypto-shlib-bss_sock.o \
	crypto/bio/libcrypto-shlib-core_bio.o \
	crypto/bn/asm/libcrypto-shlib-x86_64-gcc.o \
	crypto/bn/libcrypto-shlib-bn_add.o \
	crypto/bn/libcrypto-shlib-bn_blind.o \
	crypto/bn/libcrypto-shlib-bn_const.o \
	crypto/bn/libcrypto-shlib-bn_conv.o \
	crypto/bn/libcrypto-shlib-bn_ctx.o \
	crypto/bn/libcrypto-shlib-bn_depr.o \
	crypto/bn/libcrypto-shlib-bn_dh.o \
	crypto/bn/libcrypto-shlib-bn_div.o \
	crypto/bn/libcrypto-shlib-bn_err.o \
	crypto/bn/libcrypto-shlib-bn_exp.o \
	crypto/bn/libcrypto-shlib-bn_exp2.o \
	crypto/bn/libcrypto-shlib-bn_gcd.o \
	crypto/bn/libcrypto-shlib-bn_gf2m.o \
	crypto/bn/libcrypto-shlib-bn_intern.o \
	crypto/bn/libcrypto-shlib-bn_kron.o \
	crypto/bn/libcrypto-shlib-bn_lib.o \
	crypto/bn/libcrypto-shlib-bn_mod.o \
	crypto/bn/libcrypto-shlib-bn_mont.o \
	crypto/bn/libcrypto-shlib-bn_mpi.o \
	crypto/bn/libcrypto-shlib-bn_mul.o \
	crypto/bn/libcrypto-shlib-bn_nist.o \
	crypto/bn/libcrypto-shlib-bn_prime.o \
	crypto/bn/libcrypto-shlib-bn_print.o \
	crypto/bn/libcrypto-shlib-bn_rand.o \
	crypto/bn/libcrypto-shlib-bn_recp.o \
	crypto/bn/libcrypto-shlib-bn_rsa_fips186_4.o \
	crypto/bn/libcrypto-shlib-bn_shift.o \
	crypto/bn/libcrypto-shlib-bn_sqr.o \
	crypto/bn/libcrypto-shlib-bn_sqrt.o \
	crypto/bn/libcrypto-shlib-bn_srp.o \
	crypto/bn/libcrypto-shlib-bn_word.o \
	crypto/bn/libcrypto-shlib-bn_x931p.o \
	crypto/bn/libcrypto-shlib-rsaz-avx2.o \
	crypto/bn/libcrypto-shlib-rsaz-avx512.o \
	crypto/bn/libcrypto-shlib-rsaz-x86_64.o \
	crypto/bn/libcrypto-shlib-rsaz_exp.o \
	crypto/bn/libcrypto-shlib-rsaz_exp_x2.o \
	crypto/bn/libcrypto-shlib-x86_64-gf2m.o \
	crypto/bn/libcrypto-shlib-x86_64-mont.o \
	crypto/bn/libcrypto-shlib-x86_64-mont5.o \
	crypto/buffer/libcrypto-shlib-buf_err.o \
	crypto/buffer/libcrypto-shlib-buffer.o \
	crypto/camellia/libcrypto-shlib-cmll-x86_64.o \
	crypto/camellia/libcrypto-shlib-cmll_cfb.o \
	crypto/camellia/libcrypto-shlib-cmll_ctr.o \
	crypto/camellia/libcrypto-shlib-cmll_ecb.o \
	crypto/camellia/libcrypto-shlib-cmll_misc.o \
	crypto/camellia/libcrypto-shlib-cmll_ofb.o \
	crypto/cast/libcrypto-shlib-c_cfb64.o \
	crypto/cast/libcrypto-shlib-c_ecb.o \
	crypto/cast/libcrypto-shlib-c_enc.o \
	crypto/cast/libcrypto-shlib-c_ofb64.o \
	crypto/cast/libcrypto-shlib-c_skey.o \
	crypto/chacha/libcrypto-shlib-chacha-x86_64.o \
	crypto/cmac/libcrypto-shlib-cmac.o \
	crypto/cmp/libcrypto-shlib-cmp_asn.o \
	crypto/cmp/libcrypto-shlib-cmp_client.o \
	crypto/cmp/libcrypto-shlib-cmp_ctx.o \
	crypto/cmp/libcrypto-shlib-cmp_err.o \
	crypto/cmp/libcrypto-shlib-cmp_hdr.o \
	crypto/cmp/libcrypto-shlib-cmp_http.o \
	crypto/cmp/libcrypto-shlib-cmp_msg.o \
	crypto/cmp/libcrypto-shlib-cmp_protect.o \
	crypto/cmp/libcrypto-shlib-cmp_server.o \
	crypto/cmp/libcrypto-shlib-cmp_status.o \
	crypto/cmp/libcrypto-shlib-cmp_util.o \
	crypto/cmp/libcrypto-shlib-cmp_vfy.o \
	crypto/cms/libcrypto-shlib-cms_asn1.o \
	crypto/cms/libcrypto-shlib-cms_att.o \
	crypto/cms/libcrypto-shlib-cms_cd.o \
	crypto/cms/libcrypto-shlib-cms_dd.o \
	crypto/cms/libcrypto-shlib-cms_dh.o \
	crypto/cms/libcrypto-shlib-cms_ec.o \
	crypto/cms/libcrypto-shlib-cms_enc.o \
	crypto/cms/libcrypto-shlib-cms_env.o \
	crypto/cms/libcrypto-shlib-cms_err.o \
	crypto/cms/libcrypto-shlib-cms_ess.o \
	crypto/cms/libcrypto-shlib-cms_io.o \
	crypto/cms/libcrypto-shlib-cms_kari.o \
	crypto/cms/libcrypto-shlib-cms_lib.o \
	crypto/cms/libcrypto-shlib-cms_pwri.o \
	crypto/cms/libcrypto-shlib-cms_rsa.o \
	crypto/cms/libcrypto-shlib-cms_sd.o \
	crypto/cms/libcrypto-shlib-cms_smime.o \
	crypto/comp/libcrypto-shlib-c_zlib.o \
	crypto/comp/libcrypto-shlib-comp_err.o \
	crypto/comp/libcrypto-shlib-comp_lib.o \
	crypto/conf/libcrypto-shlib-conf_api.o \
	crypto/conf/libcrypto-shlib-conf_def.o \
	crypto/conf/libcrypto-shlib-conf_err.o \
	crypto/conf/libcrypto-shlib-conf_lib.o \
	crypto/conf/libcrypto-shlib-conf_mall.o \
	crypto/conf/libcrypto-shlib-conf_mod.o \
	crypto/conf/libcrypto-shlib-conf_sap.o \
	crypto/conf/libcrypto-shlib-conf_ssl.o \
	crypto/crmf/libcrypto-shlib-crmf_asn.o \
	crypto/crmf/libcrypto-shlib-crmf_err.o \
	crypto/crmf/libcrypto-shlib-crmf_lib.o \
	crypto/crmf/libcrypto-shlib-crmf_pbm.o \
	crypto/ct/libcrypto-shlib-ct_b64.o \
	crypto/ct/libcrypto-shlib-ct_err.o \
	crypto/ct/libcrypto-shlib-ct_log.o \
	crypto/ct/libcrypto-shlib-ct_oct.o \
	crypto/ct/libcrypto-shlib-ct_policy.o \
	crypto/ct/libcrypto-shlib-ct_prn.o \
	crypto/ct/libcrypto-shlib-ct_sct.o \
	crypto/ct/libcrypto-shlib-ct_sct_ctx.o \
	crypto/ct/libcrypto-shlib-ct_vfy.o \
	crypto/ct/libcrypto-shlib-ct_x509v3.o \
	crypto/des/libcrypto-shlib-cbc_cksm.o \
	crypto/des/libcrypto-shlib-cbc_enc.o \
	crypto/des/libcrypto-shlib-cfb64ede.o \
	crypto/des/libcrypto-shlib-cfb64enc.o \
	crypto/des/libcrypto-shlib-cfb_enc.o \
	crypto/des/libcrypto-shlib-des_enc.o \
	crypto/des/libcrypto-shlib-ecb3_enc.o \
	crypto/des/libcrypto-shlib-ecb_enc.o \
	crypto/des/libcrypto-shlib-fcrypt.o \
	crypto/des/libcrypto-shlib-fcrypt_b.o \
	crypto/des/libcrypto-shlib-ofb64ede.o \
	crypto/des/libcrypto-shlib-ofb64enc.o \
	crypto/des/libcrypto-shlib-ofb_enc.o \
	crypto/des/libcrypto-shlib-pcbc_enc.o \
	crypto/des/libcrypto-shlib-qud_cksm.o \
	crypto/des/libcrypto-shlib-rand_key.o \
	crypto/des/libcrypto-shlib-set_key.o \
	crypto/des/libcrypto-shlib-str2key.o \
	crypto/des/libcrypto-shlib-xcbc_enc.o \
	crypto/dh/libcrypto-shlib-dh_ameth.o \
	crypto/dh/libcrypto-shlib-dh_asn1.o \
	crypto/dh/libcrypto-shlib-dh_backend.o \
	crypto/dh/libcrypto-shlib-dh_check.o \
	crypto/dh/libcrypto-shlib-dh_depr.o \
	crypto/dh/libcrypto-shlib-dh_err.o \
	crypto/dh/libcrypto-shlib-dh_gen.o \
	crypto/dh/libcrypto-shlib-dh_group_params.o \
	crypto/dh/libcrypto-shlib-dh_kdf.o \
	crypto/dh/libcrypto-shlib-dh_key.o \
	crypto/dh/libcrypto-shlib-dh_lib.o \
	crypto/dh/libcrypto-shlib-dh_meth.o \
	crypto/dh/libcrypto-shlib-dh_pmeth.o \
	crypto/dh/libcrypto-shlib-dh_prn.o \
	crypto/dh/libcrypto-shlib-dh_rfc5114.o \
	crypto/dsa/libcrypto-shlib-dsa_ameth.o \
	crypto/dsa/libcrypto-shlib-dsa_asn1.o \
	crypto/dsa/libcrypto-shlib-dsa_backend.o \
	crypto/dsa/libcrypto-shlib-dsa_check.o \
	crypto/dsa/libcrypto-shlib-dsa_depr.o \
	crypto/dsa/libcrypto-shlib-dsa_err.o \
	crypto/dsa/libcrypto-shlib-dsa_gen.o \
	crypto/dsa/libcrypto-shlib-dsa_key.o \
	crypto/dsa/libcrypto-shlib-dsa_lib.o \
	crypto/dsa/libcrypto-shlib-dsa_meth.o \
	crypto/dsa/libcrypto-shlib-dsa_ossl.o \
	crypto/dsa/libcrypto-shlib-dsa_pmeth.o \
	crypto/dsa/libcrypto-shlib-dsa_prn.o \
	crypto/dsa/libcrypto-shlib-dsa_sign.o \
	crypto/dsa/libcrypto-shlib-dsa_vrf.o \
	crypto/dso/libcrypto-shlib-dso_dl.o \
	crypto/dso/libcrypto-shlib-dso_dlfcn.o \
	crypto/dso/libcrypto-shlib-dso_err.o \
	crypto/dso/libcrypto-shlib-dso_lib.o \
	crypto/dso/libcrypto-shlib-dso_openssl.o \
	crypto/dso/libcrypto-shlib-dso_vms.o \
	crypto/dso/libcrypto-shlib-dso_win32.o \
	crypto/ec/curve448/arch_32/libcrypto-shlib-f_impl32.o \
	crypto/ec/curve448/arch_64/libcrypto-shlib-f_impl64.o \
	crypto/ec/curve448/libcrypto-shlib-curve448.o \
	crypto/ec/curve448/libcrypto-shlib-curve448_tables.o \
	crypto/ec/curve448/libcrypto-shlib-eddsa.o \
	crypto/ec/curve448/libcrypto-shlib-f_generic.o \
	crypto/ec/curve448/libcrypto-shlib-scalar.o \
	crypto/ec/libcrypto-shlib-curve25519.o \
	crypto/ec/libcrypto-shlib-ec2_oct.o \
	crypto/ec/libcrypto-shlib-ec2_smpl.o \
	crypto/ec/libcrypto-shlib-ec_ameth.o \
	crypto/ec/libcrypto-shlib-ec_asn1.o \
	crypto/ec/libcrypto-shlib-ec_backend.o \
	crypto/ec/libcrypto-shlib-ec_check.o \
	crypto/ec/libcrypto-shlib-ec_curve.o \
	crypto/ec/libcrypto-shlib-ec_cvt.o \
	crypto/ec/libcrypto-shlib-ec_deprecated.o \
	crypto/ec/libcrypto-shlib-ec_err.o \
	crypto/ec/libcrypto-shlib-ec_key.o \
	crypto/ec/libcrypto-shlib-ec_kmeth.o \
	crypto/ec/libcrypto-shlib-ec_lib.o \
	crypto/ec/libcrypto-shlib-ec_mult.o \
	crypto/ec/libcrypto-shlib-ec_oct.o \
	crypto/ec/libcrypto-shlib-ec_pmeth.o \
	crypto/ec/libcrypto-shlib-ec_print.o \
	crypto/ec/libcrypto-shlib-ecdh_kdf.o \
	crypto/ec/libcrypto-shlib-ecdh_ossl.o \
	crypto/ec/libcrypto-shlib-ecdsa_ossl.o \
	crypto/ec/libcrypto-shlib-ecdsa_sign.o \
	crypto/ec/libcrypto-shlib-ecdsa_vrf.o \
	crypto/ec/libcrypto-shlib-eck_prn.o \
	crypto/ec/libcrypto-shlib-ecp_mont.o \
	crypto/ec/libcrypto-shlib-ecp_nist.o \
	crypto/ec/libcrypto-shlib-ecp_nistp224.o \
	crypto/ec/libcrypto-shlib-ecp_nistp256.o \
	crypto/ec/libcrypto-shlib-ecp_nistp521.o \
	crypto/ec/libcrypto-shlib-ecp_nistputil.o \
	crypto/ec/libcrypto-shlib-ecp_nistz256-x86_64.o \
	crypto/ec/libcrypto-shlib-ecp_nistz256.o \
	crypto/ec/libcrypto-shlib-ecp_oct.o \
	crypto/ec/libcrypto-shlib-ecp_smpl.o \
	crypto/ec/libcrypto-shlib-ecx_backend.o \
	crypto/ec/libcrypto-shlib-ecx_key.o \
	crypto/ec/libcrypto-shlib-ecx_meth.o \
	crypto/ec/libcrypto-shlib-x25519-x86_64.o \
	crypto/encode_decode/libcrypto-shlib-decoder_err.o \
	crypto/encode_decode/libcrypto-shlib-decoder_lib.o \
	crypto/encode_decode/libcrypto-shlib-decoder_meth.o \
	crypto/encode_decode/libcrypto-shlib-decoder_pkey.o \
	crypto/encode_decode/libcrypto-shlib-encoder_err.o \
	crypto/encode_decode/libcrypto-shlib-encoder_lib.o \
	crypto/encode_decode/libcrypto-shlib-encoder_meth.o \
	crypto/encode_decode/libcrypto-shlib-encoder_pkey.o \
	crypto/engine/libcrypto-shlib-eng_all.o \
	crypto/engine/libcrypto-shlib-eng_cnf.o \
	crypto/engine/libcrypto-shlib-eng_ctrl.o \
	crypto/engine/libcrypto-shlib-eng_dyn.o \
	crypto/engine/libcrypto-shlib-eng_err.o \
	crypto/engine/libcrypto-shlib-eng_fat.o \
	crypto/engine/libcrypto-shlib-eng_init.o \
	crypto/engine/libcrypto-shlib-eng_lib.o \
	crypto/engine/libcrypto-shlib-eng_list.o \
	crypto/engine/libcrypto-shlib-eng_openssl.o \
	crypto/engine/libcrypto-shlib-eng_pkey.o \
	crypto/engine/libcrypto-shlib-eng_rdrand.o \
	crypto/engine/libcrypto-shlib-eng_table.o \
	crypto/engine/libcrypto-shlib-tb_asnmth.o \
	crypto/engine/libcrypto-shlib-tb_cipher.o \
	crypto/engine/libcrypto-shlib-tb_dh.o \
	crypto/engine/libcrypto-shlib-tb_digest.o \
	crypto/engine/libcrypto-shlib-tb_dsa.o \
	crypto/engine/libcrypto-shlib-tb_eckey.o \
	crypto/engine/libcrypto-shlib-tb_pkmeth.o \
	crypto/engine/libcrypto-shlib-tb_rand.o \
	crypto/engine/libcrypto-shlib-tb_rsa.o \
	crypto/err/libcrypto-shlib-err.o \
	crypto/err/libcrypto-shlib-err_all.o \
	crypto/err/libcrypto-shlib-err_all_legacy.o \
	crypto/err/libcrypto-shlib-err_blocks.o \
	crypto/err/libcrypto-shlib-err_prn.o \
	crypto/ess/libcrypto-shlib-ess_asn1.o \
	crypto/ess/libcrypto-shlib-ess_err.o \
	crypto/ess/libcrypto-shlib-ess_lib.o \
	crypto/evp/libcrypto-shlib-asymcipher.o \
	crypto/evp/libcrypto-shlib-bio_b64.o \
	crypto/evp/libcrypto-shlib-bio_enc.o \
	crypto/evp/libcrypto-shlib-bio_md.o \
	crypto/evp/libcrypto-shlib-bio_ok.o \
	crypto/evp/libcrypto-shlib-c_allc.o \
	crypto/evp/libcrypto-shlib-c_alld.o \
	crypto/evp/libcrypto-shlib-cmeth_lib.o \
	crypto/evp/libcrypto-shlib-ctrl_params_translate.o \
	crypto/evp/libcrypto-shlib-dh_ctrl.o \
	crypto/evp/libcrypto-shlib-dh_support.o \
	crypto/evp/libcrypto-shlib-digest.o \
	crypto/evp/libcrypto-shlib-dsa_ctrl.o \
	crypto/evp/libcrypto-shlib-e_aes.o \
	crypto/evp/libcrypto-shlib-e_aes_cbc_hmac_sha1.o \
	crypto/evp/libcrypto-shlib-e_aes_cbc_hmac_sha256.o \
	crypto/evp/libcrypto-shlib-e_aria.o \
	crypto/evp/libcrypto-shlib-e_bf.o \
	crypto/evp/libcrypto-shlib-e_camellia.o \
	crypto/evp/libcrypto-shlib-e_cast.o \
	crypto/evp/libcrypto-shlib-e_chacha20_poly1305.o \
	crypto/evp/libcrypto-shlib-e_des.o \
	crypto/evp/libcrypto-shlib-e_des3.o \
	crypto/evp/libcrypto-shlib-e_idea.o \
	crypto/evp/libcrypto-shlib-e_null.o \
	crypto/evp/libcrypto-shlib-e_old.o \
	crypto/evp/libcrypto-shlib-e_rc2.o \
	crypto/evp/libcrypto-shlib-e_rc4.o \
	crypto/evp/libcrypto-shlib-e_rc4_hmac_md5.o \
	crypto/evp/libcrypto-shlib-e_rc5.o \
	crypto/evp/libcrypto-shlib-e_seed.o \
	crypto/evp/libcrypto-shlib-e_sm4.o \
	crypto/evp/libcrypto-shlib-e_xcbc_d.o \
	crypto/evp/libcrypto-shlib-ec_ctrl.o \
	crypto/evp/libcrypto-shlib-ec_support.o \
	crypto/evp/libcrypto-shlib-encode.o \
	crypto/evp/libcrypto-shlib-evp_cnf.o \
	crypto/evp/libcrypto-shlib-evp_enc.o \
	crypto/evp/libcrypto-shlib-evp_err.o \
	crypto/evp/libcrypto-shlib-evp_fetch.o \
	crypto/evp/libcrypto-shlib-evp_key.o \
	crypto/evp/libcrypto-shlib-evp_lib.o \
	crypto/evp/libcrypto-shlib-evp_pbe.o \
	crypto/evp/libcrypto-shlib-evp_pkey.o \
	crypto/evp/libcrypto-shlib-evp_rand.o \
	crypto/evp/libcrypto-shlib-evp_utils.o \
	crypto/evp/libcrypto-shlib-exchange.o \
	crypto/evp/libcrypto-shlib-kdf_lib.o \
	crypto/evp/libcrypto-shlib-kdf_meth.o \
	crypto/evp/libcrypto-shlib-kem.o \
	crypto/evp/libcrypto-shlib-keymgmt_lib.o \
	crypto/evp/libcrypto-shlib-keymgmt_meth.o \
	crypto/evp/libcrypto-shlib-legacy_blake2.o \
	crypto/evp/libcrypto-shlib-legacy_md2.o \
	crypto/evp/libcrypto-shlib-legacy_md4.o \
	crypto/evp/libcrypto-shlib-legacy_md5.o \
	crypto/evp/libcrypto-shlib-legacy_md5_sha1.o \
	crypto/evp/libcrypto-shlib-legacy_mdc2.o \
	crypto/evp/libcrypto-shlib-legacy_ripemd.o \
	crypto/evp/libcrypto-shlib-legacy_sha.o \
	crypto/evp/libcrypto-shlib-legacy_wp.o \
	crypto/evp/libcrypto-shlib-m_null.o \
	crypto/evp/libcrypto-shlib-m_sigver.o \
	crypto/evp/libcrypto-shlib-mac_lib.o \
	crypto/evp/libcrypto-shlib-mac_meth.o \
	crypto/evp/libcrypto-shlib-names.o \
	crypto/evp/libcrypto-shlib-p5_crpt.o \
	crypto/evp/libcrypto-shlib-p5_crpt2.o \
	crypto/evp/libcrypto-shlib-p_dec.o \
	crypto/evp/libcrypto-shlib-p_enc.o \
	crypto/evp/libcrypto-shlib-p_legacy.o \
	crypto/evp/libcrypto-shlib-p_lib.o \
	crypto/evp/libcrypto-shlib-p_open.o \
	crypto/evp/libcrypto-shlib-p_seal.o \
	crypto/evp/libcrypto-shlib-p_sign.o \
	crypto/evp/libcrypto-shlib-p_verify.o \
	crypto/evp/libcrypto-shlib-pbe_scrypt.o \
	crypto/evp/libcrypto-shlib-pmeth_check.o \
	crypto/evp/libcrypto-shlib-pmeth_gn.o \
	crypto/evp/libcrypto-shlib-pmeth_lib.o \
	crypto/evp/libcrypto-shlib-signature.o \
	crypto/ffc/libcrypto-shlib-ffc_backend.o \
	crypto/ffc/libcrypto-shlib-ffc_dh.o \
	crypto/ffc/libcrypto-shlib-ffc_key_generate.o \
	crypto/ffc/libcrypto-shlib-ffc_key_validate.o \
	crypto/ffc/libcrypto-shlib-ffc_params.o \
	crypto/ffc/libcrypto-shlib-ffc_params_generate.o \
	crypto/ffc/libcrypto-shlib-ffc_params_validate.o \
	crypto/hmac/libcrypto-shlib-hmac.o \
	crypto/http/libcrypto-shlib-http_client.o \
	crypto/http/libcrypto-shlib-http_err.o \
	crypto/http/libcrypto-shlib-http_lib.o \
	crypto/idea/libcrypto-shlib-i_cbc.o \
	crypto/idea/libcrypto-shlib-i_cfb64.o \
	crypto/idea/libcrypto-shlib-i_ecb.o \
	crypto/idea/libcrypto-shlib-i_ofb64.o \
	crypto/idea/libcrypto-shlib-i_skey.o \
	crypto/kdf/libcrypto-shlib-kdf_err.o \
	crypto/lhash/libcrypto-shlib-lh_stats.o \
	crypto/lhash/libcrypto-shlib-lhash.o \
	crypto/libcrypto-shlib-asn1_dsa.o \
	crypto/libcrypto-shlib-bsearch.o \
	crypto/libcrypto-shlib-context.o \
	crypto/libcrypto-shlib-core_algorithm.o \
	crypto/libcrypto-shlib-core_fetch.o \
	crypto/libcrypto-shlib-core_namemap.o \
	crypto/libcrypto-shlib-cpt_err.o \
	crypto/libcrypto-shlib-cpuid.o \
	crypto/libcrypto-shlib-cryptlib.o \
	crypto/libcrypto-shlib-ctype.o \
	crypto/libcrypto-shlib-cversion.o \
	crypto/libcrypto-shlib-der_writer.o \
	crypto/libcrypto-shlib-ebcdic.o \
	crypto/libcrypto-shlib-ex_data.o \
	crypto/libcrypto-shlib-getenv.o \
	crypto/libcrypto-shlib-info.o crypto/libcrypto-shlib-init.o \
	crypto/libcrypto-shlib-initthread.o \
	crypto/libcrypto-shlib-mem.o \
	crypto/libcrypto-shlib-mem_sec.o \
	crypto/libcrypto-shlib-o_dir.o \
	crypto/libcrypto-shlib-o_fopen.o \
	crypto/libcrypto-shlib-o_init.o \
	crypto/libcrypto-shlib-o_str.o \
	crypto/libcrypto-shlib-o_time.o \
	crypto/libcrypto-shlib-packet.o \
	crypto/libcrypto-shlib-param_build.o \
	crypto/libcrypto-shlib-param_build_set.o \
	crypto/libcrypto-shlib-params.o \
	crypto/libcrypto-shlib-params_dup.o \
	crypto/libcrypto-shlib-params_from_text.o \
	crypto/libcrypto-shlib-passphrase.o \
	crypto/libcrypto-shlib-provider.o \
	crypto/libcrypto-shlib-provider_conf.o \
	crypto/libcrypto-shlib-provider_core.o \
	crypto/libcrypto-shlib-provider_predefined.o \
	crypto/libcrypto-shlib-punycode.o \
	crypto/libcrypto-shlib-self_test_core.o \
	crypto/libcrypto-shlib-sparse_array.o \
	crypto/libcrypto-shlib-threads_lib.o \
	crypto/libcrypto-shlib-threads_none.o \
	crypto/libcrypto-shlib-threads_pthread.o \
	crypto/libcrypto-shlib-threads_win.o \
	crypto/libcrypto-shlib-trace.o crypto/libcrypto-shlib-uid.o \
	crypto/libcrypto-shlib-x86_64cpuid.o \
	crypto/md2/libcrypto-shlib-md2_dgst.o \
	crypto/md2/libcrypto-shlib-md2_one.o \
	crypto/md4/libcrypto-shlib-md4_dgst.o \
	crypto/md4/libcrypto-shlib-md4_one.o \
	crypto/md5/libcrypto-shlib-md5-x86_64.o \
	crypto/md5/libcrypto-shlib-md5_dgst.o \
	crypto/md5/libcrypto-shlib-md5_one.o \
	crypto/md5/libcrypto-shlib-md5_sha1.o \
	crypto/mdc2/libcrypto-shlib-mdc2_one.o \
	crypto/mdc2/libcrypto-shlib-mdc2dgst.o \
	crypto/modes/libcrypto-shlib-aesni-gcm-x86_64.o \
	crypto/modes/libcrypto-shlib-cbc128.o \
	crypto/modes/libcrypto-shlib-ccm128.o \
	crypto/modes/libcrypto-shlib-cfb128.o \
	crypto/modes/libcrypto-shlib-ctr128.o \
	crypto/modes/libcrypto-shlib-cts128.o \
	crypto/modes/libcrypto-shlib-gcm128.o \
	crypto/modes/libcrypto-shlib-ghash-x86_64.o \
	crypto/modes/libcrypto-shlib-ocb128.o \
	crypto/modes/libcrypto-shlib-ofb128.o \
	crypto/modes/libcrypto-shlib-siv128.o \
	crypto/modes/libcrypto-shlib-wrap128.o \
	crypto/modes/libcrypto-shlib-xts128.o \
	crypto/objects/libcrypto-shlib-o_names.o \
	crypto/objects/libcrypto-shlib-obj_dat.o \
	crypto/objects/libcrypto-shlib-obj_err.o \
	crypto/objects/libcrypto-shlib-obj_lib.o \
	crypto/objects/libcrypto-shlib-obj_xref.o \
	crypto/ocsp/libcrypto-shlib-ocsp_asn.o \
	crypto/ocsp/libcrypto-shlib-ocsp_cl.o \
	crypto/ocsp/libcrypto-shlib-ocsp_err.o \
	crypto/ocsp/libcrypto-shlib-ocsp_ext.o \
	crypto/ocsp/libcrypto-shlib-ocsp_http.o \
	crypto/ocsp/libcrypto-shlib-ocsp_lib.o \
	crypto/ocsp/libcrypto-shlib-ocsp_prn.o \
	crypto/ocsp/libcrypto-shlib-ocsp_srv.o \
	crypto/ocsp/libcrypto-shlib-ocsp_vfy.o \
	crypto/ocsp/libcrypto-shlib-v3_ocsp.o \
	crypto/pem/libcrypto-shlib-pem_all.o \
	crypto/pem/libcrypto-shlib-pem_err.o \
	crypto/pem/libcrypto-shlib-pem_info.o \
	crypto/pem/libcrypto-shlib-pem_lib.o \
	crypto/pem/libcrypto-shlib-pem_oth.o \
	crypto/pem/libcrypto-shlib-pem_pk8.o \
	crypto/pem/libcrypto-shlib-pem_pkey.o \
	crypto/pem/libcrypto-shlib-pem_sign.o \
	crypto/pem/libcrypto-shlib-pem_x509.o \
	crypto/pem/libcrypto-shlib-pem_xaux.o \
	crypto/pem/libcrypto-shlib-pvkfmt.o \
	crypto/pkcs12/libcrypto-shlib-p12_add.o \
	crypto/pkcs12/libcrypto-shlib-p12_asn.o \
	crypto/pkcs12/libcrypto-shlib-p12_attr.o \
	crypto/pkcs12/libcrypto-shlib-p12_crpt.o \
	crypto/pkcs12/libcrypto-shlib-p12_crt.o \
	crypto/pkcs12/libcrypto-shlib-p12_decr.o \
	crypto/pkcs12/libcrypto-shlib-p12_init.o \
	crypto/pkcs12/libcrypto-shlib-p12_key.o \
	crypto/pkcs12/libcrypto-shlib-p12_kiss.o \
	crypto/pkcs12/libcrypto-shlib-p12_mutl.o \
	crypto/pkcs12/libcrypto-shlib-p12_npas.o \
	crypto/pkcs12/libcrypto-shlib-p12_p8d.o \
	crypto/pkcs12/libcrypto-shlib-p12_p8e.o \
	crypto/pkcs12/libcrypto-shlib-p12_sbag.o \
	crypto/pkcs12/libcrypto-shlib-p12_utl.o \
	crypto/pkcs12/libcrypto-shlib-pk12err.o \
	crypto/pkcs7/libcrypto-shlib-bio_pk7.o \
	crypto/pkcs7/libcrypto-shlib-pk7_asn1.o \
	crypto/pkcs7/libcrypto-shlib-pk7_attr.o \
	crypto/pkcs7/libcrypto-shlib-pk7_doit.o \
	crypto/pkcs7/libcrypto-shlib-pk7_lib.o \
	crypto/pkcs7/libcrypto-shlib-pk7_mime.o \
	crypto/pkcs7/libcrypto-shlib-pk7_smime.o \
	crypto/pkcs7/libcrypto-shlib-pkcs7err.o \
	crypto/poly1305/libcrypto-shlib-poly1305-x86_64.o \
	crypto/poly1305/libcrypto-shlib-poly1305.o \
	crypto/property/libcrypto-shlib-defn_cache.o \
	crypto/property/libcrypto-shlib-property.o \
	crypto/property/libcrypto-shlib-property_err.o \
	crypto/property/libcrypto-shlib-property_parse.o \
	crypto/property/libcrypto-shlib-property_string.o \
	crypto/rand/libcrypto-shlib-prov_seed.o \
	crypto/rand/libcrypto-shlib-rand_deprecated.o \
	crypto/rand/libcrypto-shlib-rand_err.o \
	crypto/rand/libcrypto-shlib-rand_lib.o \
	crypto/rand/libcrypto-shlib-rand_meth.o \
	crypto/rand/libcrypto-shlib-rand_pool.o \
	crypto/rand/libcrypto-shlib-randfile.o \
	crypto/rc2/libcrypto-shlib-rc2_cbc.o \
	crypto/rc2/libcrypto-shlib-rc2_ecb.o \
	crypto/rc2/libcrypto-shlib-rc2_skey.o \
	crypto/rc2/libcrypto-shlib-rc2cfb64.o \
	crypto/rc2/libcrypto-shlib-rc2ofb64.o \
	crypto/rc4/libcrypto-shlib-rc4-md5-x86_64.o \
	crypto/rc4/libcrypto-shlib-rc4-x86_64.o \
	crypto/ripemd/libcrypto-shlib-rmd_dgst.o \
	crypto/ripemd/libcrypto-shlib-rmd_one.o \
	crypto/rsa/libcrypto-shlib-rsa_ameth.o \
	crypto/rsa/libcrypto-shlib-rsa_asn1.o \
	crypto/rsa/libcrypto-shlib-rsa_backend.o \
	crypto/rsa/libcrypto-shlib-rsa_chk.o \
	crypto/rsa/libcrypto-shlib-rsa_crpt.o \
	crypto/rsa/libcrypto-shlib-rsa_depr.o \
	crypto/rsa/libcrypto-shlib-rsa_err.o \
	crypto/rsa/libcrypto-shlib-rsa_gen.o \
	crypto/rsa/libcrypto-shlib-rsa_lib.o \
	crypto/rsa/libcrypto-shlib-rsa_meth.o \
	crypto/rsa/libcrypto-shlib-rsa_mp.o \
	crypto/rsa/libcrypto-shlib-rsa_mp_names.o \
	crypto/rsa/libcrypto-shlib-rsa_none.o \
	crypto/rsa/libcrypto-shlib-rsa_oaep.o \
	crypto/rsa/libcrypto-shlib-rsa_ossl.o \
	crypto/rsa/libcrypto-shlib-rsa_pk1.o \
	crypto/rsa/libcrypto-shlib-rsa_pmeth.o \
	crypto/rsa/libcrypto-shlib-rsa_prn.o \
	crypto/rsa/libcrypto-shlib-rsa_pss.o \
	crypto/rsa/libcrypto-shlib-rsa_saos.o \
	crypto/rsa/libcrypto-shlib-rsa_schemes.o \
	crypto/rsa/libcrypto-shlib-rsa_sign.o \
	crypto/rsa/libcrypto-shlib-rsa_sp800_56b_check.o \
	crypto/rsa/libcrypto-shlib-rsa_sp800_56b_gen.o \
	crypto/rsa/libcrypto-shlib-rsa_x931.o \
	crypto/rsa/libcrypto-shlib-rsa_x931g.o \
	crypto/seed/libcrypto-shlib-seed.o \
	crypto/seed/libcrypto-shlib-seed_cbc.o \
	crypto/seed/libcrypto-shlib-seed_cfb.o \
	crypto/seed/libcrypto-shlib-seed_ecb.o \
	crypto/seed/libcrypto-shlib-seed_ofb.o \
	crypto/sha/libcrypto-shlib-keccak1600-x86_64.o \
	crypto/sha/libcrypto-shlib-sha1-mb-x86_64.o \
	crypto/sha/libcrypto-shlib-sha1-x86_64.o \
	crypto/sha/libcrypto-shlib-sha1_one.o \
	crypto/sha/libcrypto-shlib-sha1dgst.o \
	crypto/sha/libcrypto-shlib-sha256-mb-x86_64.o \
	crypto/sha/libcrypto-shlib-sha256-x86_64.o \
	crypto/sha/libcrypto-shlib-sha256.o \
	crypto/sha/libcrypto-shlib-sha3.o \
	crypto/sha/libcrypto-shlib-sha512-x86_64.o \
	crypto/sha/libcrypto-shlib-sha512.o \
	crypto/siphash/libcrypto-shlib-siphash.o \
	crypto/sm2/libcrypto-shlib-sm2_crypt.o \
	crypto/sm2/libcrypto-shlib-sm2_err.o \
	crypto/sm2/libcrypto-shlib-sm2_key.o \
	crypto/sm2/libcrypto-shlib-sm2_sign.o \
	crypto/sm3/libcrypto-shlib-legacy_sm3.o \
	crypto/sm3/libcrypto-shlib-sm3.o \
	crypto/sm4/libcrypto-shlib-sm4.o \
	crypto/srp/libcrypto-shlib-srp_lib.o \
	crypto/srp/libcrypto-shlib-srp_vfy.o \
	crypto/stack/libcrypto-shlib-stack.o \
	crypto/store/libcrypto-shlib-store_err.o \
	crypto/store/libcrypto-shlib-store_init.o \
	crypto/store/libcrypto-shlib-store_lib.o \
	crypto/store/libcrypto-shlib-store_meth.o \
	crypto/store/libcrypto-shlib-store_register.o \
	crypto/store/libcrypto-shlib-store_result.o \
	crypto/store/libcrypto-shlib-store_strings.o \
	crypto/ts/libcrypto-shlib-ts_asn1.o \
	crypto/ts/libcrypto-shlib-ts_conf.o \
	crypto/ts/libcrypto-shlib-ts_err.o \
	crypto/ts/libcrypto-shlib-ts_lib.o \
	crypto/ts/libcrypto-shlib-ts_req_print.o \
	crypto/ts/libcrypto-shlib-ts_req_utils.o \
	crypto/ts/libcrypto-shlib-ts_rsp_print.o \
	crypto/ts/libcrypto-shlib-ts_rsp_sign.o \
	crypto/ts/libcrypto-shlib-ts_rsp_utils.o \
	crypto/ts/libcrypto-shlib-ts_rsp_verify.o \
	crypto/ts/libcrypto-shlib-ts_verify_ctx.o \
	crypto/txt_db/libcrypto-shlib-txt_db.o \
	crypto/ui/libcrypto-shlib-ui_err.o \
	crypto/ui/libcrypto-shlib-ui_lib.o \
	crypto/ui/libcrypto-shlib-ui_null.o \
	crypto/ui/libcrypto-shlib-ui_openssl.o \
	crypto/ui/libcrypto-shlib-ui_util.o \
	crypto/whrlpool/libcrypto-shlib-wp-x86_64.o \
	crypto/whrlpool/libcrypto-shlib-wp_dgst.o \
	crypto/x509/libcrypto-shlib-by_dir.o \
	crypto/x509/libcrypto-shlib-by_file.o \
	crypto/x509/libcrypto-shlib-by_store.o \
	crypto/x509/libcrypto-shlib-pcy_cache.o \
	crypto/x509/libcrypto-shlib-pcy_data.o \
	crypto/x509/libcrypto-shlib-pcy_lib.o \
	crypto/x509/libcrypto-shlib-pcy_map.o \
	crypto/x509/libcrypto-shlib-pcy_node.o \
	crypto/x509/libcrypto-shlib-pcy_tree.o \
	crypto/x509/libcrypto-shlib-t_crl.o \
	crypto/x509/libcrypto-shlib-t_req.o \
	crypto/x509/libcrypto-shlib-t_x509.o \
	crypto/x509/libcrypto-shlib-v3_addr.o \
	crypto/x509/libcrypto-shlib-v3_admis.o \
	crypto/x509/libcrypto-shlib-v3_akeya.o \
	crypto/x509/libcrypto-shlib-v3_akid.o \
	crypto/x509/libcrypto-shlib-v3_asid.o \
	crypto/x509/libcrypto-shlib-v3_bcons.o \
	crypto/x509/libcrypto-shlib-v3_bitst.o \
	crypto/x509/libcrypto-shlib-v3_conf.o \
	crypto/x509/libcrypto-shlib-v3_cpols.o \
	crypto/x509/libcrypto-shlib-v3_crld.o \
	crypto/x509/libcrypto-shlib-v3_enum.o \
	crypto/x509/libcrypto-shlib-v3_extku.o \
	crypto/x509/libcrypto-shlib-v3_genn.o \
	crypto/x509/libcrypto-shlib-v3_ia5.o \
	crypto/x509/libcrypto-shlib-v3_info.o \
	crypto/x509/libcrypto-shlib-v3_int.o \
	crypto/x509/libcrypto-shlib-v3_ist.o \
	crypto/x509/libcrypto-shlib-v3_lib.o \
	crypto/x509/libcrypto-shlib-v3_ncons.o \
	crypto/x509/libcrypto-shlib-v3_pci.o \
	crypto/x509/libcrypto-shlib-v3_pcia.o \
	crypto/x509/libcrypto-shlib-v3_pcons.o \
	crypto/x509/libcrypto-shlib-v3_pku.o \
	crypto/x509/libcrypto-shlib-v3_pmaps.o \
	crypto/x509/libcrypto-shlib-v3_prn.o \
	crypto/x509/libcrypto-shlib-v3_purp.o \
	crypto/x509/libcrypto-shlib-v3_san.o \
	crypto/x509/libcrypto-shlib-v3_skid.o \
	crypto/x509/libcrypto-shlib-v3_sxnet.o \
	crypto/x509/libcrypto-shlib-v3_tlsf.o \
	crypto/x509/libcrypto-shlib-v3_utf8.o \
	crypto/x509/libcrypto-shlib-v3_utl.o \
	crypto/x509/libcrypto-shlib-v3err.o \
	crypto/x509/libcrypto-shlib-x509_att.o \
	crypto/x509/libcrypto-shlib-x509_cmp.o \
	crypto/x509/libcrypto-shlib-x509_d2.o \
	crypto/x509/libcrypto-shlib-x509_def.o \
	crypto/x509/libcrypto-shlib-x509_err.o \
	crypto/x509/libcrypto-shlib-x509_ext.o \
	crypto/x509/libcrypto-shlib-x509_lu.o \
	crypto/x509/libcrypto-shlib-x509_meth.o \
	crypto/x509/libcrypto-shlib-x509_obj.o \
	crypto/x509/libcrypto-shlib-x509_r2x.o \
	crypto/x509/libcrypto-shlib-x509_req.o \
	crypto/x509/libcrypto-shlib-x509_set.o \
	crypto/x509/libcrypto-shlib-x509_trs.o \
	crypto/x509/libcrypto-shlib-x509_txt.o \
	crypto/x509/libcrypto-shlib-x509_v3.o \
	crypto/x509/libcrypto-shlib-x509_vfy.o \
	crypto/x509/libcrypto-shlib-x509_vpm.o \
	crypto/x509/libcrypto-shlib-x509cset.o \
	crypto/x509/libcrypto-shlib-x509name.o \
	crypto/x509/libcrypto-shlib-x509rset.o \
	crypto/x509/libcrypto-shlib-x509spki.o \
	crypto/x509/libcrypto-shlib-x509type.o \
	crypto/x509/libcrypto-shlib-x_all.o \
	crypto/x509/libcrypto-shlib-x_attrib.o \
	crypto/x509/libcrypto-shlib-x_crl.o \
	crypto/x509/libcrypto-shlib-x_exten.o \
	crypto/x509/libcrypto-shlib-x_name.o \
	crypto/x509/libcrypto-shlib-x_pubkey.o \
	crypto/x509/libcrypto-shlib-x_req.o \
	crypto/x509/libcrypto-shlib-x_x509.o \
	crypto/x509/libcrypto-shlib-x_x509a.o \
	providers/libcrypto-shlib-baseprov.o \
	providers/libcrypto-shlib-defltprov.o \
	providers/libcrypto-shlib-nullprov.o \
	providers/libcrypto-shlib-prov_running.o \
	providers/libimplementations.a providers/libcommon.a providers/libnonfips.a  -ldl -pthread 
rm -f libcrypto.so && \
ln -s libcrypto.so.3 libcrypto.so
ccache clang -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -L. -Wl,-znodelete -shared -Wl,-Bsymbolic   -Wl,-soname=libssl.so.3 \
	-o libssl.so.3 -Wl,--version-script=libssl.ld \
	crypto/libssl-shlib-packet.o ssl/libssl-shlib-bio_ssl.o \
	ssl/libssl-shlib-d1_lib.o ssl/libssl-shlib-d1_msg.o \
	ssl/libssl-shlib-d1_srtp.o ssl/libssl-shlib-methods.o \
	ssl/libssl-shlib-pqueue.o ssl/libssl-shlib-s3_cbc.o \
	ssl/libssl-shlib-s3_enc.o ssl/libssl-shlib-s3_lib.o \
	ssl/libssl-shlib-s3_msg.o ssl/libssl-shlib-ssl_asn1.o \
	ssl/libssl-shlib-ssl_cert.o ssl/libssl-shlib-ssl_ciph.o \
	ssl/libssl-shlib-ssl_conf.o ssl/libssl-shlib-ssl_err.o \
	ssl/libssl-shlib-ssl_err_legacy.o \
	ssl/libssl-shlib-ssl_init.o ssl/libssl-shlib-ssl_lib.o \
	ssl/libssl-shlib-ssl_mcnf.o ssl/libssl-shlib-ssl_rsa.o \
	ssl/libssl-shlib-ssl_rsa_legacy.o \
	ssl/libssl-shlib-ssl_sess.o ssl/libssl-shlib-ssl_stat.o \
	ssl/libssl-shlib-ssl_txt.o ssl/libssl-shlib-ssl_utst.o \
	ssl/libssl-shlib-t1_enc.o ssl/libssl-shlib-t1_lib.o \
	ssl/libssl-shlib-t1_trce.o ssl/libssl-shlib-tls13_enc.o \
	ssl/libssl-shlib-tls_depr.o ssl/libssl-shlib-tls_srp.o \
	ssl/record/libssl-shlib-dtls1_bitmap.o \
	ssl/record/libssl-shlib-rec_layer_d1.o \
	ssl/record/libssl-shlib-rec_layer_s3.o \
	ssl/record/libssl-shlib-ssl3_buffer.o \
	ssl/record/libssl-shlib-ssl3_record.o \
	ssl/record/libssl-shlib-ssl3_record_tls13.o \
	ssl/record/libssl-shlib-tls_pad.o \
	ssl/statem/libssl-shlib-extensions.o \
	ssl/statem/libssl-shlib-extensions_clnt.o \
	ssl/statem/libssl-shlib-extensions_cust.o \
	ssl/statem/libssl-shlib-extensions_srvr.o \
	ssl/statem/libssl-shlib-statem.o \
	ssl/statem/libssl-shlib-statem_clnt.o \
	ssl/statem/libssl-shlib-statem_dtls.o \
	ssl/statem/libssl-shlib-statem_lib.o \
	ssl/statem/libssl-shlib-statem_srvr.o \
	-lcrypto  -ldl -pthread 
rm -f libssl.so && \
ln -s libssl.so.3 libssl.so
ccache clang -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -L. -Wl,-z,defs -Wl,-znodelete -shared -Wl,-Bsymbolic   \
	-o engines/afalg.so -Wl,--version-script=engines/afalg.ld \
	engines/afalg-dso-e_afalg.o \
	-lcrypto -ldl -pthread 
ccache clang -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -L. -Wl,-z,defs -Wl,-znodelete -shared -Wl,-Bsymbolic   \
	-o engines/capi.so -Wl,--version-script=engines/capi.ld \
	engines/capi-dso-e_capi.o \
	-lcrypto -ldl -pthread 
ccache clang -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -L. -Wl,-z,defs -Wl,-znodelete -shared -Wl,-Bsymbolic   \
	-o engines/dasync.so -Wl,--version-script=engines/dasync.ld \
	engines/dasync-dso-e_dasync.o \
	-lcrypto -ldl -pthread 
ccache clang -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -L. -Wl,-z,defs -Wl,-znodelete -shared -Wl,-Bsymbolic   \
	-o engines/loader_attic.so -Wl,--version-script=engines/loader_attic.ld \
	crypto/pem/loader_attic-dso-pvkfmt.o \
	engines/loader_attic-dso-e_loader_attic.o \
	-lcrypto -ldl -pthread 
ccache clang -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -L. -Wl,-z,defs -Wl,-znodelete -shared -Wl,-Bsymbolic   \
	-o engines/ossltest.so -Wl,--version-script=engines/ossltest.ld \
	engines/ossltest-dso-e_ossltest.o \
	-lcrypto -ldl -pthread 
ccache clang -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -L. -Wl,-z,defs -Wl,-znodelete -shared -Wl,-Bsymbolic   \
	-o engines/padlock.so -Wl,--version-script=engines/padlock.ld \
	engines/padlock-dso-e_padlock-x86_64.o \
	engines/padlock-dso-e_padlock.o \
	-lcrypto -ldl -pthread 
ccache clang -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -Wl,-z,defs -Wl,-znodelete -shared -Wl,-Bsymbolic   \
	-o providers/fips.so -Wl,--version-script=providers/fips.ld \
	providers/fips/fips-dso-fipsprov.o \
	providers/fips/fips-dso-self_test.o \
	providers/fips/fips-dso-self_test_kats.o \
	providers/libimplementations.a providers/libcommon.a providers/libfips.a -ldl -pthread 
ccache clang -fPIC -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -L. -Wl,-z,defs -Wl,-znodelete -shared -Wl,-Bsymbolic   \
	-o providers/legacy.so -Wl,--version-script=providers/legacy.ld \
	providers/legacy-dso-legacyprov.o \
	providers/liblegacy.a providers/libcommon.a providers/libnonfips.a -lcrypto -ldl -pthread 
/usr/bin/perl apps/progs.pl -C "apps/openssl" > apps/progs.c
/usr/bin/perl apps/progs.pl -H "apps/openssl" > apps/progs.h
ccache clang  -Iapps -I. -Iinclude -Iapps/include  -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_BUILDING_OPENSSL  -MMD -MF apps/openssl-bin-asn1pars.d.tmp -MT apps/openssl-bin-asn1pars.o -c -o apps/openssl-bin-asn1pars.o apps/asn1pars.c
ccache clang  -Iapps -I. -Iinclude -Iapps/include  -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_BUILDING_OPENSSL  -MMD -MF apps/openssl-bin-ca.d.tmp -MT apps/openssl-bin-ca.o -c -o apps/openssl-bin-ca.o apps/ca.c
ccache clang  -Iapps -I. -Iinclude -Iapps/include  -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_BUILDING_OPENSSL  -MMD -MF apps/openssl-bin-ciphers.d.tmp -MT apps/openssl-bin-ciphers.o -c -o apps/openssl-bin-ciphers.o apps/ciphers.c
ccache clang  -Iapps -I. -Iinclude -Iapps/include  -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_BUILDING_OPENSSL  -MMD -MF apps/openssl-bin-cmp.d.tmp -MT apps/openssl-bin-cmp.o -c -o apps/openssl-bin-cmp.o apps/cmp.c
ccache clang  -Iapps -I. -Iinclude -Iapps/include  -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_BUILDING_OPENSSL  -MMD -MF apps/openssl-bin-cmp_mock_srv.d.tmp -MT apps/openssl-bin-cmp_mock_srv.o -c -o apps/openssl-bin-cmp_mock_srv.o apps/cmp_mock_srv.c
ccache clang  -Iapps -I. -Iinclude -Iapps/include  -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_BUILDING_OPENSSL  -MMD -MF apps/openssl-bin-cms.d.tmp -MT apps/openssl-bin-cms.o -c -o apps/openssl-bin-cms.o apps/cms.c
ccache clang  -Iapps -I. -Iinclude -Iapps/include  -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_BUILDING_OPENSSL  -MMD -MF apps/openssl-bin-crl.d.tmp -MT apps/openssl-bin-crl.o -c -o apps/openssl-bin-crl.o apps/crl.c
ccache clang  -Iapps -I. -Iinclude -Iapps/include  -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_BUILDING_OPENSSL  -MMD -MF apps/openssl-bin-crl2p7.d.tmp -MT apps/openssl-bin-crl2p7.o -c -o apps/openssl-bin-crl2p7.o apps/crl2p7.c
ccache clang  -Iapps -I. -Iinclude -Iapps/include  -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_BUILDING_OPENSSL  -MMD -MF apps/openssl-bin-dgst.d.tmp -MT apps/openssl-bin-dgst.o -c -o apps/openssl-bin-dgst.o apps/dgst.c
ccache clang  -Iapps -I. -Iinclude -Iapps/include  -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_BUILDING_OPENSSL  -MMD -MF apps/openssl-bin-dhparam.d.tmp -MT apps/openssl-bin-dhparam.o -c -o apps/openssl-bin-dhparam.o apps/dhparam.c
ccache clang  -Iapps -I. -Iinclude -Iapps/include  -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_BUILDING_OPENSSL  -MMD -MF apps/openssl-bin-dsa.d.tmp -MT apps/openssl-bin-dsa.o -c -o apps/openssl-bin-dsa.o apps/dsa.c
ccache clang  -Iapps -I. -Iinclude -Iapps/include  -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_BUILDING_OPENSSL  -MMD -MF apps/openssl-bin-dsaparam.d.tmp -MT apps/openssl-bin-dsaparam.o -c -o apps/openssl-bin-dsaparam.o apps/dsaparam.c
ccache clang  -Iapps -I. -Iinclude -Iapps/include  -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_BUILDING_OPENSSL  -MMD -MF apps/openssl-bin-ec.d.tmp -MT apps/openssl-bin-ec.o -c -o apps/openssl-bin-ec.o apps/ec.c
^CMakefile:23961: recipe for target 'apps/openssl-bin-ec.o' failed
make[1]: *** [apps/openssl-bin-ec.o] Interrupt
Makefile:3315: recipe for target 'build_sw' failed
make: *** [build_sw] Interrupt

        exit sigint
; mk    
make depend && make _build_sw
make[1]: Entering directory '/home/rsalz/openssl/github'
make[1]: Leaving directory '/home/rsalz/openssl/github'
make[1]: Entering directory '/home/rsalz/openssl/github'
ccache clang  -Iapps -I. -Iinclude -Iapps/include  -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_BUILDING_OPENSSL  -MMD -MF apps/openssl-bin-ec.d.tmp -MT apps/openssl-bin-ec.o -c -o apps/openssl-bin-ec.o apps/ec.c
ccache clang  -Iapps -I. -Iinclude -Iapps/include  -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_BUILDING_OPENSSL  -MMD -MF apps/openssl-bin-ecparam.d.tmp -MT apps/openssl-bin-ecparam.o -c -o apps/openssl-bin-ecparam.o apps/ecparam.c
ccache clang  -Iapps -I. -Iinclude -Iapps/include  -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_BUILDING_OPENSSL  -MMD -MF apps/openssl-bin-enc.d.tmp -MT apps/openssl-bin-enc.o -c -o apps/openssl-bin-enc.o apps/enc.c
apps/enc.c:148:17: error: enumeration value 'OPT_EOF' not handled in switch [-Werror,-Wswitch]
        switch (o) {
                ^
1 error generated.
Makefile:23977: recipe for target 'apps/openssl-bin-enc.o' failed
make[1]: *** [apps/openssl-bin-enc.o] Error 1
make[1]: Leaving directory '/home/rsalz/openssl/github'
Makefile:3315: recipe for target 'build_sw' failed
make: *** [build_sw] Error 2
        exit 2
; clang --version
clang version 7.0.0-3~ubuntu0.18.04.1 (tags/RELEASE_700/final)
Target: x86_64-pc-linux-gnu
Thread model: posix
InstalledDir: /usr/bin
; sed -n -e 148,152p apps/enc.c 
        switch (o) {
        case OPT_ERR:
 opthelp:
            BIO_printf(bio_err, "%s: Use -help for summary.\n", prog);
            goto end;
; 
```

Reply:
(deleted old version of this comment)
```
ccache clang  -Iapps -I. -Iinclude -Iapps/include  -pthread -m64 -Wa,--noexecstack -Qunused-arguments -Wall -O0 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-unknown-warning-option -Wswitch-default -Wno-parentheses-equality -Wno-language-extension-token -Wno-extended-offsetof -Wconditional-uninitialized -Wincompatible-pointer-types-discards-qualifiers -Wmissing-variable-declarations -DOPENSSL_BUILDING_OPENSSL  -MMD -MF apps/openssl-bin-enc.d.tmp -MT apps/openssl-bin-enc.o -c -o apps/openssl-bin-enc.o apps/enc.c
apps/enc.c:148:17: error: enumeration value 'OPT_EOF' not handled in switch [-Werror,-Wswitch]
        switch (o) {
                ^
1 error generated.
Makefile:23977: recipe for target 'apps/openssl-bin-enc.o' failed
make[1]: *** [apps/openssl-bin-enc.o] Error 1
make[1]: Leaving directory '/home/rsalz/openssl/github'
Makefile:3315: recipe for target 'build_sw' failed
make: *** [build_sw] Error 2
        exit 2
; clang --version
clang version 7.0.0-3~ubuntu0.18.04.1 (tags/RELEASE_700/final)
Target: x86_64-pc-linux-gnu
Thread model: posix
InstalledDir: /usr/bin
; sed -n -e 148,152p apps/enc.c 
        switch (o) {
        case OPT_ERR:
 opthelp:
            BIO_printf(bio_err, "%s: Use -help for summary.\n", prog);
            goto end;
; 
```